# Cationic Dendrimers for siRNA Delivery: An Overview of Methods for In Vitro/In Vivo Characterization

Erik Laurini, Suzana Aulic, Domenico Marson, Maurizio Fermeglia and Sabrina Pricl

## Abstract

This chapter reviews the different techniques for analyzing the chemical-physical properties, transfection efficiency, cytotoxicity, and stability of covalent cationic dendrimers (CCDs) and self-assembled cationic dendrons (ACDs) for siRNA delivery in the presence and absence of their nucleic cargos. On the basis of the reported examples, a standard essential set of techniques is described for each step of a siRNA/nanovector (NV) complex characterization process: (1) analysis of the basic chemical-physical properties of the NV per se; (2) characterization of the morphology, size, strength, and stability of the siRNA/NV ensemble; (3) characterization and quantification of the cellular uptake and release of the siRNA fragment; (4) in vitro and (5) in vivo experiments for the evaluation of the corresponding gene silencing activity; and (6) assessment of the intrinsic toxicity of the NV and the siRNA/NV complex.

## 1 Introduction

Thanks to our deeper understanding of the genetic aberrations involved in many human pathologies, the delivery of nucleic acid therapeutics to downregulate or replace mutated genes or to suppress unwanted gene expression is becoming a highly attractive approach in nanomedicine [[1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR1)]. Small interfering RNAs (siRNAs) represent a new approach toward the inhibition of gene expression; as such, they have rapidly emerged as promising therapeutics for a plethora of important diseases including cancer, cardiovascular diseases, and other disorders of a genetic etiology [[2](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR2)–[4](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR4)]. However, the clinical translation of RNA interference (RNAi) requires safe and efficient vectors for siRNA delivery into cells [[5](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR5)]; the ability of targeting specific cells becomes a key requirement for RNAi-based therapeutics. Indeed, naked siRNAs are not amenable to therapeutic administration, as after intravenous administration these exogenous nucleic acid fragments are either readily digested by nucleases or eliminated from the kidney glomeruli before reaching the disease-afflicted organ. Furthermore, the passage of siRNAs through the cell membrane is difficult to achieve as a result of their large size and polyanionic nature. Although viral vectors are generally very efficient transfecting agents, they raise many biosafety concerns [[6](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR6)] and this, in turn, spurs the search for nonviral (i.e., synthetic) nanovectors (NVs) [[7](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR7), [8](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR8)]. Among the various nonviral carriers, cationic dendrimer NVs are emerging as the most promising ones, as they are able to generate stable nanoassemblies with siRNA through electrostatic interaction, thereby protecting the nucleic acid fragment from degradation and promoting cellular uptake [[9](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR9)]. So far, a multitude of NVs have been investigated for siRNA delivery, including poly(amidoamine) (PAMAM) dendrimers [[10](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR10)–[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14)], poly(propylene imine) (PPI) dendrimers [[15](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR15), [16](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR16)], poly(l-lysine) dendrimers [[17](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR17), [18](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR18)], and carbosilane dendrimers [[19](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR19)–[21](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR21)]. Structurally, covalent cationic dendrimers (CCDs) share a common molecular skeleton constituted by three distinct domains (Fig. [1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig1)): (1) a group of central atoms defined as the core; (2) the branching units defining the dendrimer growth through different generations (Gs); and (3) an exponentially increasing number of positive charged terminal groups. Among CCDs, PAMAM dendrimers undoubtedly constitute the molecules most widely explored as siRNA delivery NVs since they are provided with several beneficial features [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14), [22](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR22)].

![485053_1_En_14_Fig1_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig1_HTML.png)

**Fig. 1** Cartoon representation of a CCD structure highlighting its three, distinct structural motifs: the core (light blue), the branching units forming the different G generation (spring green), and the cationic terminal groups (dark blue)

One of the main issues concerning CCDs is that only the highest generations (e.g., G ≧ 5) are provided with efficient and effective siRNA transfecting agents. However, synthesizing a large amount of pure, high-G CCDs involves considerable costs, time, and efforts [[23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23), [24](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR24)]. To overcome this limitation, a new approach has been recently developed based on the exploitation of self-assembly, of a quintessential nanotechnology method. Accordingly, different amphiphilic cationic dendrons (ACDs) have been designed and synthetized. ACDs couple the multivalent cooperativity of highest generations of CCD NVs with the self-assembling property of lipid vectors (Fig. [2](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig2)) [[23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23), [25](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR25)].

![485053_1_En_14_Fig2_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig2_HTML.png)

**Fig. 2** Cartoon representation of an ACD structure. The branching units (spring green) and the cationic terminal groups (dark blue) represent the hydrophilic portion of a CCD structure while the black lines identify its lipophilic moiety

Self-assembly is a process that relies on the cumulative effects of multiple non-covalent interactions to assemble ACD building blocks into supramolecular entities in a reversible, controllable, and specific way. Consequently, ACDs are able to auto-organize themselves into nanosized micelles which, in most instances, outperform their covalent dendrimer counterparts for siRNA delivery purposes [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR26)–[28](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR28)].

Both CCDs and ACDs require a variety of adequate experimental techniques for their characterization. Specifically, the complex pathway of a siRNA/NV nanoparticle toward its target cells must be validated in all its steps (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig3)): from the characterization of its peculiar properties (I) and the assessment of stability of the intermolecular complex (II) to the siRNA transport across the cell membrane and the delivery of the nucleic acid to its proper location (III) through interactions with the intracellular trafficking machinery. The silencing activity of the siRNA/NV complex must be finally validated at cellular level (IV, in vitro) and then translated to animal models (V, in vivo), again through the exploitation of specific and robust experimental approaches. Lastly, the intrinsic toxicity profile (VI) of the nanocarrier and its complex with siRNA has to be investigated for its eventual therapeutic translation.

![485053_1_En_14_Fig3_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig3_HTML.png)

**Fig. 3** Schematic representation of each experimental validation phase of a siRNA/NV system

In this chapter we attempt to systemize these techniques for each characterization step and briefly describe their specific requirements and outputs according to our experience acquired in the last decade about the design, synthesis, and characterization of PAMAM-based CCDs and ACDs.

Figure [4](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig4) shows the structures of the nanovectors developed by our research group, which will be adopted in the article as proof-of-principle to illustrate the various techniques used for their characterization and the corresponding achievements [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14), [23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23)]. The information reported in this chapter are, however, of general validity; therefore, they can be used by any research groups in the quest of safer and more efficient nanovectors for gene therapy.

![485053_1_En_14_Fig4_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig4_HTML.png)

**Fig. 4** Structure of the CCDs and ACDs used in the chapter as proof-of-principle siRNA NVs

## 2 Chemical-Physical Characterization of the Pure NV

For the effective exploitation of RNAi, every siRNA delivery system must be small and stable since small nanoparticulate complexes can penetrate deep into their target tissues. Furthermore, small and stable nanosystems can be efficiently taken up by cells and effectively protect the siRNA against enzyme degradation [[29](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR29)]. Accordingly, experimental techniques that yield information about the morphology and dimensions of the NVs alone assume a pivotal role. Moreover, the knowledge of particle size distribution and polydispersity index (PDI) of cationic NVs is also of outmost importance, as these are two main physicochemical attributes that influence their endocytosis-dependent cellular uptake. In addition, cationic NVs interact with their cargo through ionic bonding that originates at their interface with the oppositely charged nucleic acid fragments. On the other hand, it is known that molecules with high positive charge densities are potentially endowed with elevated toxicity due to their unspecific interaction with cell membranes [[30](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR30)]. Accordingly, the characterization and the eventual subsequent fine-tuning of the charge distribution on a NV surface is of paramount importance to achieve an optimal balance between the required siRNA affinity and a low-toxicity profile.

Finally, in the case of ACD NVs , it is essential to analyze in detail the thermodynamics of their self-aggregation process, and to characterize the stability of the relevant nanoassemblies by deriving fundamental chemical-physical parameters. One of such quantities is the critical micellar concentration (CMC), which is defined as the concentration of ACDs in solution above which ACDs aggregate into micelles.

### 2.1 Dynamic Light Scattering (DLS)

Detection of light scattering from matter is a useful technique with applications in numerous scientific disciplines where, depending on the light source and detector, specific molecular properties can be studied. In a typical DLS experiment, the sample under investigation is exposed to a monochromatic wave of light (typically a laser). Over the course of the analysis, the laser beam, scattered in all directions upon colliding with the particles in solution, is captured by a detector [[31](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR31)]. Accordingly, DLS is widely used for studying the structure and the dimensions of NVs [[32](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR32), [33](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR33)]. In general, the technique is best used for submicron particles although in principle it can be used to measure particle with sizes less than a nanometer. In DLS, the time-dependent fluctuations in the scattered light are measured by a fast photon counter. These fluctuations are directly related to the rate of diffusion of the particle through the solvent, which is related in turn to their hydrodynamic radii. The particle diffusion coefficient (*D*) is calculated by fitting the correlation curve to an exponential function, with *D* being proportional to the lifetime of the exponential decay. Once known, *D* is then used to calculate the particle hydrodynamic diameter (*D*h) through the Stokes–Einstein equation [[32](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR32)]:

$$ D=\frac{kT}{3\pi \eta {D}_{\mathrm{h}}} $$

where *k* is the Boltzmann constant, *T* is the temperature, and *η* is the medium viscosity. With small differences, in most cases the hydrodynamic radius (*R*h = *D*h/2) derived by DLS is comparable with the gyration radius (*R*g)—i.e., the average distance of any atom in a molecule from its center of mass—obtainable from other experimental techniques such as small angle neutron scattering (SANS) and small angle X-ray scattering (SAXS) [[34](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR34), [35](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR35)]. Therefore, both *D*h and *R*g can provide information about the actual dimensions of a NV.

In DLS another important output is PDI, a number calculated from a simple two-parameter fit to the correlation data (the cumulants analysis). PDI is dimensionless and describes the broadness of the particle size distribution, ranging from 0 (theoretical) for a perfectly mono-disperse sample to 1 for a sample with a very broad size distribution containing multiple particle sizes. Anyhow, values smaller than 0.05 are rarely seen other than with highly monodisperse standards and values above 0.4 indicate that the sample has a very broad size distribution and is likely not suitable for most applications, nanomedicine in particular [[33](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR33)].

Finally, a further important parameter that can be measured by DLS is zeta-potential (*ζ*). By definition, this quantity provides a measure of the electrostatic potential between the bulk solvent and the stationary layer of fluid attached to a dispersed particle and, as such, is related to the particle surface charge [[33](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR33)]. Many colloidal or other nanoparticle dispersions bear surface charges once in solution. If an electric field ***E*** is applied to these systems, the underlying electrostatic interactions cause the particles to jiggle in a motion whose direction and velocity are governed by the particle charge itself, the nature of the solvent, and the intensity of the applied ***E***. Thus, DLS-based laser Doppler electrophoresis can be employed to measure the small frequency shifts in the scattered light that arise owing to these ***E***-induced particle movements which, in turn, are linked to *ζ* (in mV) via different theoretical expression, the most common being the Smulochowski model [[36](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR36)].

Figure [5a, b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig5) illustrates representative examples of NV DLS analysis ; in particular, the PAMAM-ACD-1 nanocarrier is provided with a low PDI value of 0.10 and it predominantly forms nanoparticles around 7.0 nm in size with a positive *ζ*-potential of +35 mV [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR26), [27](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR27)].

![485053_1_En_14_Fig5_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig5_HTML.png)

**Fig. 5** DLS analysis of D*h* (**a**) and *ζ*-potential (**b**) and TEM images (**c**) of spherical micelles formed by PAMAM-ACD-1. (Adapted from [[27](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR27)] with the permission of John Wiley and Sons)

### 2.2 Transmission Electron Microscopy (TEM)

Different types of microscopy techniques are used for NVs imaging such as TEM, atomic force microscopy (AFM), and polarized optical microscopy [[37](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR37), [38](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR38)]. In TEM, an electron beam passing through an ultra-thin specimen is absorbed and scattered, producing contrast and an image. Because of the very short wavelength of the electron beam employed, TEM can achieve sub-nanometer resolutions well below that of even the highest-resolution light microscopes (∼20 nm) [[39](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR39), [40](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR40)]. The general procedure for the preparation of a TEM sample requires the use of an ultramicrotome in order to obtain the thinnest possible sample allowing for effective electron transmission through it [[41](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR41)]. Working with nanoscale materials such as CCDs or ACDs somewhat simplifies this process which, however, remains troublesome and time-consuming [[42](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR42)]. Indeed, dendrimer-based sample preparation for TEM imaging involves the deposition of the material onto extra-thin carbon-coated copper support grids, followed by sample drying in air [[43](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR43)]. The subsequent stage commonly consists in grid staining with heavy metal salt solutions (typically uranyl acetate or lead citrate), a process required in order to enhance the contrast between the nanostructures under observation and the background [[43](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR43), [44](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR44)].

An example of TEM imaging obtained for cationic dendrimer-based NVs is shown in Fig. [5c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig5): the PAMAM-ACD-1 nanoassemblies appear as white objects on a dark background of the amorphous carbon substrate. If, as in the present case, the visualized nanoobjects are well separated from each other, the measurement of their sizes and their morphology description can easily be accomplished using an image analysis software. In the specific case of PAMAM-ACD-1 NVs, TEM imaging is in agreement with the corresponding DLS data (panels a and b), since it reveals the formation of small spherical particles with sizes in the range of typical micellar nanoobjects (6–7 nm) [[27](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR27)].

### 2.3 Isothermal Titration Calorimetry (ITC)

Understanding in detail the self-assembling process of ACDs for siRNA delivery is important because the specific micellar phase of a given ACD may influence its biological effects [[45](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR45)]; moreover, the knowledge of the chemical details of these aggregation processes could allow for precise control of siRNA transport. ITC is a reliable, simple, and rapid technique for obtaining the full set of thermodynamic parameters that drive micelle formation and determine the CMC value for any amphiphilic molecule. Although a plethora of different techniques is available to experimentally determine CMC values—ranging from spectroscopic [[46](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR46)] to electrochemical [[47](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR47)], and optical [[48](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR48)] measurements—only ITC is able, through the determination of the whole thermodynamic process associated with underlying self-assembly phenomenon, to provide information for designing and obtaining more stable and better performing ACD systems [[49](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR49)]. In combination with the structural evidence provided by DLS and TEM, ITC data complete the set of information required for a complete description of ACD NVs micellization. As described in detail below, somewhat counterintuitively ITC experiments describe a demicellization process, which is the reverse phenomenon of the corresponding self-assembly. Accordingly, Fig. [6](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6) shows the demicellization thermogram of the ACD-3 NV [[50](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR50)] in buffer solution as an example. ITC experiments carried out for ACD-3 can be used as prototypical examples of how an ACD micellization process could be described through calorimetric measurements. At the beginning of the titration (Fig. [6b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6)), the injection of a concentrated ACD-3 solution into a buffer solution mostly results in large, endothermic (i.e., positive) signals, followed by a series of very small, exothermic (negative) peaks in the latest stages of the experiment (Fig. [6c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6)). The enthalpy change characterizing the first part of the thermogram corresponds to the progressive addition of a concentrated ACD-3 solution (i.e., *C* > CMC) to the buffer (sample cell), resulting in the subsequent ACD-3 micelle dilution, ACD-3 demicellization, and dilution of the ACD-3 monomers. When more solution of ACD-3 micelles is titrated into the sample cell, a clear decrease in the heat is observed, indicating that the added micelles do no longer dissociate or, in other words, the CMC value is attained. In the final part of the thermogram, further additions of the concentrated ACD-3 micellar solution to the sample cell leads to heat effects due to micellar dilution, being the ACD-3 concentration in the cell well above the corresponding CMC. To obtain the corresponding numerical CMC value, ITC raw data be further elaborated as follows: (1) subtraction of the micelle dilution heat must be performed; (2) data normalization per mol of ACD-3 must be carried out, (3) the resulting heat raw data must be integrated to yield the heat *Q* as a function of the concentration *C* curve shown in Fig. [6b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6) and, finally, (4) the CMC can be estimated as the midpoint of this *Q* vs. *C* curve. However, for a more precise determination of the CMC value, *Q* vs. *C* data can be fitted to a suitable model from which the corresponding first derivative can easily be calculated (*see* Fig. [6d](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6)) (both these last two processes can be carried out with simple versatile statistic tools, e.g., GraphPad). Accordingly, the CMC value corresponds to the minimum of this derivative curve, as highlighted by the arrow in Fig. [6d](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6). From the ITC titration curve, the micellization enthalpy (Δ*H*mic) is calculated from the enthalpy difference of the two levels of the titration curve as shown in Fig. [6e](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6) [[49](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR49)]. The experimental determination of the Δ*H*mic and the CMC values then allows for the calculation of the other thermodynamic parameters. The free energy of micellization (Δ*G*mic) can be approximated by the expression Δ*G*mic = *RT* ln CMC′, where *R* is the gas constant (8.31e−3 kJ/mol K) and *T* is the absolute temperature and CMC′ is the critical micellization concentration expressed in its molar units. The change in entropy associated with the micellization (*T*Δ*S*mic) can be calculated from the second law of thermodynamics by using the Gibbs–Helmholtz equation *T*Δ*S*mic = Δ*H*mic − Δ*G*mic. As an example, all the thermodynamic parameters obtained for the micellization of the ACD-3 NV are listed in Fig. [6f](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig6). As it can be seen, the entropy contribution dominates the micellization process, with the enthalpy playing a minor role (|*T*Δ*S*mic| ≫ |Δ*H*mic|) and this can be specifically related to the releasing of the water molecules in the hydration shells around the hydrophobic parts of the ACD-3 monomers with the subsequent production of hydrophobic forces [[50](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR50)].

![485053_1_En_14_Fig6_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig6_HTML.png)

**Fig. 6** (**a**) Structure of compound ACD-3 . (**b**) Measured heat power vs. time elapsed during the titration. (**c**) Heat observed on each injection vs. final ACD-3 concentration in the calorimetric cell. Symbols: experimental data; red line: data fitting with a Boltzmann sigmoid model (*R*2 = 0.9953). (**d**) The first derivative of the curve in **c** in arbitrary units. The CMC is taken as the *x*-value at the minimum of d*Q*/d*C* curve (indicated by the arrow). (**e**) Determination of the Δ*H*mic obtained from the integrated ITC data. (**f**) CMC value and thermodynamics parameter for the micellization of ACD-3. (Adapted from [[50](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR50)] with permission of the RSC)

## 3 siRNA/NV Complex Characterization

The ability of a nanocarrier to generate a stable complex with its nucleic acid cargo is a background postulate for its usefulness in gene delivery. A deep understanding about dimensions and morphology of the siRNA/NV complex is a key point for the experimental characterization. In line with what reported above, DLS-based quantities such as *R*H, *ζ*, and PDI coupled with TEM imaging can provide both qualitative and quantitative basic information also in the case of siRNA/NV complexes. Regarding the binding affinity between NV and siRNA, this property can be experimentally explored and quantified through the use of many different techniques, ranging from displacement assays (e.g., the standard Ethidium Bromide (EthBr) displacement fluorescence spectroscopy assay [[51](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR51)]) to the more sophisticate and quantitative ITC [[9](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR9)]. Also, the agarose gel retardation assay can provide further knowledge about a siRNA/NV complex formation and stability [[52](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR52)]. It is worth to mention here that one of the most important physicochemical parameters for characterizing any given cationic siRNA/NV complex is the so-called ± ratio, that is the total number of positive charges on the cationic nanovector (+) divided by the total number of negative charges carried by the nucleic acid fragment (−). In the specific case of PAMAM-based cationic dendrimer, this ratio is defined as *N*/*P* (with *N* = number of dendrimer positively-charged nitrogen atoms and *P* = number of siRNA phosphate (P) groups), and it is used in place of the more standard concentration ratio in both in vitro and in vivo characterization and activity determination of these nanocarrier/siRNA systems.

### 3.1 DLS and TEM

DLS (Subheading [2.1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec3)) and TEM (Subheading [2.2](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec4)) are also widely employed for characterizing size and morphology of siRNA/NV complexes. As briefly discussed above, any ideal siRNA/NV complex should be characterized by dimension laying on the nanometric scale in order to display optimal cellular uptake and bioavailability. Concomitantly, the *ζ*-potential of these nanoassemblies should be high enough to ensure colloidal stability but also low enough to limit the related cytotoxic effects [[53](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR53)]. Figure [7](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig7) shows the TEM and DLS results obtained for the nanocomplex formed between PAMAM-ACD-2 and siRNA as an example. These siRNA/NV nanoparticles are uniformly spherical in shape (Fig. [7a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig7)), with a diameter of ∼37 nm and characterized by a narrow PDI value of 0.20 (Fig. [7b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig7)). In addition, their *ζ* value is around +15 mV (Fig. [7c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig7)), implying stable colloidal property [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR54)].

![485053_1_En_14_Fig7_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig7_HTML.png)

**Fig. 7** (**a**) TEM imaging , (**b**) DLS and (**c**) *ζ*-potential analysis of the siRNA/ACD-2 complexes at an *N*/*P* ratio of 10. (Adapted from [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR54)] with permission of FSG)

### 3.2 ITC

The determination of the affinity of a given NV for a siRNA molecule is an exquisite example of ITC native function, i.e., measuring the heat change during a binding interaction between two molecular entities [[55](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR55)]. Indeed, ITC is the only experimental technique that can directly capture the full binding thermodynamics of diverse biological interactions including protein-ligand binding [[56](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR56)], protein-protein binding [[57](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR57)], DNA-protein binding [[58](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR58)] and, of course, nucleic acid-NV binding. ITC has the ability to precisely determine the change in Gibbs free energy (Δ*G*) and in its enthalpic (Δ*H*) and entropic (Δ*S*) components associated with binding. Figure [8](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8) shows the raw calorimetric thermogram and the corresponding integrated values for an ITC experiment performed in order to determine the affinity of the PAMAM-ACD-2 NV for a siRNA [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR54)]. Basically, each injection of the siRNA solution into the sample cell containing a concentrated solution of the NV (*C* > CMC) triggers the relevant binding reaction, and the formation of the nanoassembly is accompanied by the release of heat (exothermic reaction). Hence, during the titration the detected signals are recorded as typical calorimetric peaks (Fig. [8a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8)) which, upon integration, provide the amount of heat associated with each injection (Fig. [8b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8)). As the NV micelles become saturated with siRNA fragments, the heat signal progressively diminishes until only the background heat, due to unspecific phenomena, is observed. With a procedure utterly similar to that discussed in Subheading [2.3](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec5), the complete set of the thermodynamic functions (Δ*G*, Δ*H*, and *T*Δ*S*) governing the binding interaction between the NV and the siRNA can be obtained from the data in Fig. [8b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8).

![485053_1_En_14_Fig8_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig8_HTML.png)

**Fig. 8** Isothermal titration calorimetry raw (**a**) and integrated (**b**) data for titration of PAMAM-ACD-2 with siRNA in water (*T* = 25 °C). The dotted red line represents data fitting with a sigmoidal function. (Adapted from [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR54)] with permission of FSG)

In the example illustrated in Fig. [8](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8), the siRNA/PAMAM-ACD-2 complex exhibits a thermodynamically spontaneous supported by a largely negative (i.e., favorable) value of the corresponding binding free energy (Δ*G* = −9.78 kcal/mol). The driving force underlying the high NV affinity for its siRNA is prevalently enthalpic (Δ*H* = −6.53 kcal/mol), reflecting the essentially electrostatic nature of the siRNA/PAMAM-ACD-2 complex interaction (Fig. [8](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig8)). The related entropy variation is also favorable (i.e., positive, *T*Δ*S* = 3.25 kcal/mol), and can be ascribed to the release of water and counter ions from the siRNA/PAMAM-ACD-2 binding interface into bulk solvent.

### 3.3 Agarose Gel Retardation Analysis and RNAse A Assay

Agarose gel electrophoresis is a useful technique to study the separation of nucleic acid fragments. It is also a simple, low cost, and rapid approach to monitor the complex formation between siRNA and NV and stability of siRNA/NV complexes in the presence of RNA-degrading enzymes such as RNases [[11](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR11), [59](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR59)]. The optimal composition for this analysis is a 1.2% of agarose in standard Tris-borate-EDTA buffer (TBE), and run time generally is about 15 min. The nucleic acid bands can be stained by EthBr or by the less toxic SYBR Green dye [[52](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR52)]. In the case of CCD, for the agarose gel retardation analysis is important to remind that the complex formation depends on the dendrimer generation [[60](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR60)]. Specifically, the positive end groups of a CCD can completely retard nucleic acid in the gels at high *N*/*P* ratios. In other words, higher CCD Gs are provided with the strongest interaction for nucleic acids, and hence the formed complex is more stable (Fig. [9](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig9)).

![485053_1_En_14_Fig9_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig9_HTML.png)

**Fig. 9** Formation of siRNA/PAMAM-CCD-G5 complexes revealed by agarose gel retardation analysis. This indicates that dendrimers of higher generations form more stable complexes, presumably due to their greater cooperativity between the -NH3+ end groups on the dendrimer and PO43− groups on the nucleic acid. (Adapted from [[60](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR60)] with permission of ACS)

In addition, in order to study the stability of the siRNA/NV complexes a treatment with fixed concentration of RNase at incremental steps is commonly used (e.g., 0.25 μg/μL of RNase A for 0, 5, 10, 15, 20, 30, 45, 60, 75, 90, and 120 min at 37 °C). The RNase A enzyme in an endoribonuclease that specifically degrades single-stranded RNA at C and U residues [[52](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR52)]. As reported in the Fig. [10](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig10) [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)], the degradation by RNase enzyme is prevented in the case of strong binding that occurs between siRNA molecules and their NVs. The final result indicates that PAMAM-CCD-G5 complexation effectively shields the siRNA fragments and protect them from degradation compared to naked siRNAs, for which the enzymatic digestion by the RNase is observed immediately (after 5 min). This implies that complexes are stable and are able to protect siRNA from degradation, independently of the nucleic acid length (25-mer or 27-mer, respectively).

![485053_1_En_14_Fig10_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig10_HTML.png)

**Fig. 10** The nanocomplex with PAMAM-CCD-G5 protects siRNA (**a**) 25-mer or (**b**) 27-mer from RNase degradation. Compared to the naked siRNA, the nucleic acid molecule complexed with PAMAM-CCD-G5 is resistant to RNase digestion and remains stable even after 120 min incubation. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)] with permission of ACS)

## 4 siRNA Cellular Uptake and Release

One of the crucial steps of the experimental characterization of NVs for siRNA delivery is verifying that the NV, once complexed with its cargo, is able to promote cellular uptake. Subsequently, the complex has to be able to perform endosomal escape into the cell cytoplasm, thereby releasing the siRNA and making it readily accessible to the cellular RNAi machinery for effective gene silencing. Along this complex pathway, the first important issues to check are whether the transported siRNA is able to specifically reach and if it can be internalized by the desired target cells. These requirements may be enabled using two strategies: passive and active targeting. The first approach relies on the (still controversial) intrinsic enhanced permeability and retention effect (EPR) [[62](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR62)], according to which molecules with size in the nanometer range tend to accumulate in diseased tissues much more than they do in normal tissues. This is of particular relevance when exploiting RNAi in cancer treatment; here, the EPR effect refers to the preferential accumulation of nanoparticles within tumors owing to their leaky vasculature and poor lymphatic drainage. On the other hand, active targeting employs functionalization or directed intermolecular interactions to confer higher specificity to a delivery system. Among others, the most exploited targeting groups for NV are peptides [[63](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR63)], folates [[64](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR64)], carbohydrates [[65](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR65)], and monoclonal antibodies [[66](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR66)]. In this regard, it is essential to understand not only the effectiveness of recognition and uptake but also the precise underlying mechanism. Whether passive or active NV targeting strategies are employed, the verification of the uptake and the release of the siRNA/NV nanoassemblies into the cytoplasm must be carried out before any other evaluation of the silencing effectiveness.

### 4.1 Flow Cytometry (FC)

The current technological innovations in flow cytometry techniques actually permit their use in the study of cellular uptake of siRNA/NV complexes at the single cell level [[67](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR67)]. FC is predominantly used to measure fluorescence intensity produced by siRNA/NV complexes tagged with fluorescent labels [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14), [23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23), [67](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR67), [68](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR68)]; however, appropriate cell models and target genes have to be used. In this context, we briefly discuss below our study on RNAi performed in human prostate cancer cell line (PC-3) using a Cy5-labeled siRNA that targets the cancer cell survival gene *HSP27* (Heat Shock Protein gene 27). Specifically, our group reported that decoration of the siRNA/PAMAM-ACD-2 nanosystem with the targeting peptide E16G6RGDK results in its optimal cellular uptake in this cell line (Fig. [11a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig11)) [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)]. Moreover, the presence of this targeting peptide on the surface of the siRNA/PAMAM-ACD-2 assembly improves its delivery efficiency and specificity via a dual binding to ανβ3 integrin (overexpressed in tumor vasculature) and neuropilin-1 (a receptor present on tumor cells). Indeed, the internalization of the peptide-coated Cy5-siRNA/PAMAM-ACD-2/E16G6RGDK complexes in PC-3 cells is significantly higher (3×) than the corresponding noncoated Cy5-siRNA/PAMAM-ACD-2 complexes (Fig. [11b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig11)). In addition, the uptake of the Cy5-siRNA/PAMAM-ACD-2/E16G6RGDK complexes is considerably reduced in the presence of a specific ανβ3 integrin binder such as the cyclic RGD (cRGD) peptide (Fig. [11b, c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig11)). This clearly demonstrates that cRGD competes with the E16G6RGDK-coated siRNA/NV complexes for integrin binding via specific interaction with the integrin-binding RGD motif.

![485053_1_En_14_Fig11_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig11_HTML.png)

**Fig. 11**(**a**) structure of the targeting peptide  E16G6RGDK (**b**) FC analysis of the cellular uptake of siRNA/PAMAM-ACD-2/E16G6RGDK complexes compared to siRNA/PAMAM-ACD-2 complexes using Cy5-labeled siRNA. (**c**) FC analysis of the cellular uptake of siRNA/PAMAM-ACD-2/E16G6RGDK complexes in the absence and presence of cyclic RGD (cRGD). (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

### 4.2 Confocal Microscopy

The characterization of siRNA uptake and release can be further validated and/or expanded by combining FC-based analyses with an integrative approach such as confocal microscopy (CM) [[70](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR70)]. Indeed, CM permits the visual localization of the same fluorescently labeled complexes used in FC experiments. Additionally, CM co-localization analysis allows to determine whether two or more different molecules locate in the same cellular compartment [[71](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR71)]. Figure [12](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig12) reports an example of the CM analysis for the cellular uptake of the siRNA/PAMAM-ACD-2/E16G6RGDK complex discussed above [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)]. As it can be observed in Fig. [12](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig12), pretreating PC-3 cells with an anti neuropilin-1 (Nrp-1) receptor antibody dramatically reduces the internalization of the Cy5-siRNA/PAMAM-ACD-2/E16G6RGDK complex compared to antibody-untreated cells, thereby confirming that the postulated second mechanism of cellular uptake—i.e., via Nrp-1 receptor interaction—is also effectively in place for this targeted nanoassembly.

![485053_1_En_14_Fig12_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig12_HTML.png)

**Fig. 12** CM imaging of the cellular uptake of siRNA/PAMAM-ACD-2 complexes with the targeting peptide E16G6RGDK in the absence and presence of a Nrp-1-receptor antibody. (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

CM can also be used to investigate different cellular process dynamics in living cells [[71](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR71)]. A representative example in the NV-mediated siRNA delivery field is reported in the study discussed below [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)], in which we used far-red fluorescent dye Alexa647-labeled siRNAs [[72](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR72)] to transfect PC-3 cells in presence or absence of PAMAM-CCD-G5 NVs. The results shown in Fig. [13a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13) indicate that PAMAM-CCD-G5 is indeed quite effective in mediating siRNA cellular uptake.

![485053_1_En_14_Fig13_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig13_HTML.png)

**Fig. 13** Uptake of Alexa647-labeled siRNA/PAMAM-CCD-G5 complexes on PC-3 cells analyzed by live-cell confocal microscopy (**a**), using specific inhibitors (**b**) and fluorescent endocytic biomarkers (**c**) of different uptake pathways and phalloidin probe for actin rearrangement (**d**). (**a**) Blue channel image of the nuclei of PC-3 cells stained by Hoechst 34580 (left), red channel image of the Alexa647-labeled siRNA/PAMAM-CCD-G5 nanoparticles (red), and merged image of both (right). (**b**) Effect of cytochalasin D (to inhibit macropinocytosis), genistein (to inhibit caveolae-mediated endocytosis), and chlorpromazine (to inhibit clathrin-mediated endocytosis) on the cell uptake of the Alexa647-labeled siRNA/G5 complexes. (**c**) Colocalization of Alexa647-labeled siRNA/PAMAM-CCD-G5 complexes with different endocytic markers: dextran (marker of macropinocytosis), cholera toxin B (marker of caveolae-mediated endocytosis), and transferrin (marker of clathrin-mediated endocytosis), green channel image of the Alexa488-labeled different endocytic biomarkers. (**d**) Fluorescent labeling of actin fibers using Alexa647-labeled phalloidin reveals actin depolymerization, a hallmark of macropinocytosis. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)] with permission of ACS)

The different mechanisms governing the cellular uptake of siRNA/NV nano-ensembles (e.g., macropinocytosis, clathrin-mediated endocytosis, and caveolae-mediated endocytosis) can also be identified by CM [[73](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR73)]. For instance, each endocytosis pathway can be individually and selectively investigated by using specific inhibitors and/or specific endocytic markers. As a proof-of-concept, to determine the mechanism underlying the cell internalization of the siRNA/PAMAM-CCD-G5 complexes in Fig. [13](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13) we employed cytochalasin D as a macropinocytosis inhibitor [[74](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR74)], genistein to inhibit caveolae-mediated endocytosis [[75](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR75)], and chlorpromazine [[76](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR76)] to block chlatrin-mediated uptake (Fig. [13b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13)). In the same context, we also used CM to eventually co-localize the internalized siRNA/PAMAM-CCD-G5 nanoparticles with dextran [[77](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR77)], cholera toxin B [[78](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR78)], and transferrin [[79](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR79)] as markers of the three main endocytosis pathways, respectively (Fig. [13c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13)). Concerning the use of endocytosis inhibitors, from Fig. [13b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13) it is clearly seen that a significant reduction in the cellular uptake of the siRNA/PAMAM-CCD-G5 complexes is observed only when the cells are treated with cytochalasin D, leading to the conclusion that macropinocytosis is the prevailing internalization mechanism for these supermolecular nanoassemblies (Fig. [13b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13)). In line, the Alexa647 labeled siRNA/PAMAM-CCD-G5 complexes are mainly found to co-localize with the macropinocytosis marker dextran, whereas only a minor colocalization is detected with the markers of the two alternative uptake pathways (Fig. [13c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13)). As a final observation, significant actin depolymerization—an additional hallmark of macropinocytosis—is also observed after siRNA/PAMAM-CCD-G5 complex internalization (Fig. [13d](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig13)).

Once siRNA/NV complexes are internalized into target cells, their endosome entrapment and the subsequent successful escape into the cytoplasm are two fundamental stages in the complex RNAi process, which can still be investigated via CM using, e.g., LysoTracker [[80](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR80)], which fluorescently labels acidic compartments [[81](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR81), [82](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR82)]. The siRNA/PAMAM-ACD-2/E16G6RGDK nanosystem presented above (Figs. [11](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig11) and [12](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig12)) offers a good example in this respect [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)]. Indeed, as seen in Fig. [14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig14) these Cy5-labeled nucleic acid/NV targeted nanoparticles completely co-localize with the LysoTracker marker already 30 min post PC-3 cell incubation (p.c.i.), indicating their complete endosome entrapment. Importantly, however, after another 30′ (1-h total) this colocalization is less evident, suggesting that some of the nanocomplexes have effectively escaped from the endosomes into the cytoplasm [[83](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR83)]. The almost quantitative endosomal escape is further confirmed by CM imaging performed 2 h p.c.i., in which the fluorescent signals from the siRNA-containing nanoassemblies are mainly diffused through the cell cytoplasm.

![485053_1_En_14_Fig14_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig14_HTML.png)

**Fig. 14** Confocal microscopic imaging of the endosomal escape of the siRNA/PAMAM-ACD-2/E16G6RGDK complexes after incubation times of 0 (control), 0.5, 1.0, and 2.0 h. Red channel image shows the Cy5-labeled siRNA/PAMAM-ACD-2/E16G6RGDK complexes, green channel image shows the endosomes marked by LysoTracker red, and blue channel image shows the nuclei of PC-3 cells stained by Hoechst 33342 [[83](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR83)]. (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

## 5 In Vitro Silencing Activity

As it is well known, NVs are used in order to quantitatively transport and delivery siRNAs that, by silencing specific disease-related genes, ultimately suppress the corresponding mRNA expression and, hence, the synthesis of the related encoded protein(s) [[84](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR84)–[87](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR87)]. As our own experience in the field mainly refers to NV-assisted RNAi in cancer, and prostate cancer (PC) in particular, this specific application will be used as a proof-of-concept in this section. PC is one of the most common cancers in industrialized countries, and anticancer treatments based on molecular targeting of survival genes using RNAi can constitute a novel and promising approach for this pathology [[88](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR88)]. As mentioned in Subheading [4.1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec11), the *HSP27* gene and its encoded Hsp27 protein (an ATP-independent molecular chaperone upregulated by hormonal ablation and chemotherapy) is now recognized as an important therapeutic target in prostate cancer [[89](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR89)] for which a specific targeted siRNA has been recently developed [[90](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR90), [91](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR91)]. Accordingly, we investigated the ability of the different cationic NVs characterized in the previous sections to elicit RNAi in PC both in vitro and in vivo. The techniques employed in these investigations constitute the topic of the last part of this chapter.

### 5.1 Quantitative Real-Time Polymerase Chain Reaction (qRT-PCR)

(qRT)-PCR is the standard technique that quantifies nucleic acids. Accordingly, it is routinely used to determine the levels of residual mRNA expression after the corresponding specific gene silencing via RNAi. In brief, this quantification methodology measures the changes in gene expression in a sample of interest relative to a reference sample. The protocol usually involves three main steps, that is (1) a reverse transcription step in which the RNA is converted into a complementary DNA copy (cDNA), (2) several rounds of cDNA amplification, and finally (3) the detection and quantification of the amplified products in real time [[92](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR92), [93](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR93)]. Figure [15a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig15) reports an example of qRT-PCR results showing how a *HSP27*-targeted siRNA delivered by the PAMAM-CCD-G5 NV results in effective and specific gene silencing and anticancer activity in a prostate cancer PC-3 cell model [[12](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR12), [61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)]. As it can be seen, the levels of the *HSP27* mRNA expression are reduced by approximately 75% when the *HSP27* targeting siRNA is delivered by the cationic dendrimer NV with respect to untreated cells. Further, this potent gene silencing effect is specific, since cell transfection with a scramble siRNA delivered by the same NV (i.e., a siRNA with a random sequence) or the NV alone do not produce any noticeable effect. Panel b in Fig. [15](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig15) shows another aspect of RNAi obtained with the same *HSP27* siRNA yet delivered to PC-3 cells via the PAMAM-ACD-2 NV, both in the presence and in the absence of the E16G6RGDK targeting peptide (*see* Subheadings [4.1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec11) and [4.2](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec12)). Indeed, from this figure it can be observed how the presence of the targeting moieties on the surface of the ACD-2 NVs results in a significant increase of *HSP27* silencing at the corresponding mRNA level compared to the non-coated siRNA/PAMAM-ACD-2 complexes (Fig. [15b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig15)).

![485053_1_En_14_Fig15_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig15_HTML.png)

**Fig. 15** Targeted gene silencing effect determined using qRT-PCR show effective and specific gene silencing and anticancer activity in prostate cancer PC-3 cells. (**a**) *HSP27* siRNA/PAMAM-CCD-G5 complex mediated downregulation of *HSP27* mRNA expression. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)] with permission of ACS). (**b**) *HSP27* silencing at mRNA level with siRNA/PAMAM-ACD-2/E16G6RGDK shows how the complex system is much more effective than the un-coated siRNA/PAMAM-ACD-2 system for *HSP27* silencing and antiproliferation in PC-3 prostate cancer cells. (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

### 5.2 Western Blotting (WB)

WB is a prototypical molecular biology technique used to detect and identify specific proteins of interest in a sample of tissue homogenate or extract. Typically, WB first require protein separation by gel electrophoresis—a method for the separation and analysis of macromolecules including DNA, RNA , and proteins and their fragments, based on their size and charge. The gel (usually Sodium Dodecyl Sulfate polyacrylamide gel (SDS-PAGE)) is then placed next to a nitrocellulose or PVDF (polyvinylidene fluoride) membrane, and the application of an electric field causes the proteins to migrate from the gel to the membrane. Finally, the membrane is probed by antibodies targeting the proteins under investigation, and visualized using secondary antibodies and detection reagents. As such, WB is normally used in tandem with (qRT)-PCR to evaluate RNAi efficiency at the protein level [[94](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR94)]. For the purpose, a housekeeping protein that is constitutively expressed at high levels in most cells and tissues is used as positive loading control [[95](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR95), [96](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR96)] and for the relative quantification of the protein(s) under study (via the ratio of the intensity of each protein WB band relative to that of the control). In addition, non-transfected cells and/or cells transfected with another non-targeting siRNA complex are also analyzed as negative controls. Finally, Fig. [16](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig16) shows the WB results for the *HSP27* targeting siRNA delivered by the same NV systems discussed above, i.e., PAMAM-CCD-G and PAMAM-ACD-2 [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61), [69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)]. As shown in this figure panels, the gene silencing results determined at the mRNA by (qRT)-PCR (Fig. [15](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig15)) are confirmed by the corresponding WB evidences at the protein level, as a significant suppression of the Hsp27 protein expression is achieved via siRNA delivery promoted both by the covalent PAMAM-CCD-G5 NV (Fig. [16a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig16)) and by the peptide-coated PAMAM-ACD-2 nanocarrier (Fig. [16b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig16)).

![485053_1_En_14_Fig16_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig16_HTML.png)

**Fig. 16** Hsp27 protein expression after silencing with the corresponding siRNA dendrimer complexes in human prostate PC-3 cells, determined by western blotting technique for (**a**) siRNA/PAMAM-CCD-G5 and (**b**) siRNA/PAMAM-ACD-2 nanocomplexes. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61), [69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

### 5.3 MTT Assay

With specific reference to RNAi in cancer therapeutics, quite often the genes targeted for silencing play a major role in the regulation of cellular apoptosis, i.e., a form of programmed cell death. As mentioned above, *HSP27* is one of such genes, as it is upregulated in cancer cells and plays a significant role in apoptosis inhibition [[89](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR89)–[91](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR91)]. Accordingly, once its siRNA/NV-mediated silencing has been verified, the evaluation of the related effect on cancer cell growth constitutes another fundamental step. The 3-(4,5-dimethythiazol2-yl)-2,5-diphenyl tetrazolium bromide (MTT) assay is currently the most commonly used method to test cytotoxicity effects on cellular viability [[97](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR97)]. This is an assay that measures the color change from the yellow MTT to the dark purple insoluble formazan upon reduction by mitochondrial succinate dehydrogenase. This process occurs only in metabolically active cells; thus, the level of mitochondrial activity is a convenient marker of proliferation of viable cells. Figure [17](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig17) reports the results of MTT assays performed on PC-3 cells after treatment with *HSP27*-directed siRNA in complex with the PAMAM-CCD-G5 (Fig. [17a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig17)) or with PAMAM-ACD-2 (Fig. [17b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig17)) NVs, respectively [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61), [69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)], from which the relevant significant inhibition of cancer cell proliferation upon RNAi can be readily observed. Also, the specificity of the cytotoxic effect can be readily appreciated, as cell transfection with a scrambled siRNA or with the NVs alone indeed results in cell viability comparable to that of the control (untreated cells).

![485053_1_En_14_Fig17_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig17_HTML.png)

**Fig. 17** MTT assay of the antiproliferation effect of the *HSP27* targeting siRNA in complex with PAMAM-CCD-G5 (**a**) and with PAMAM-ACD-2 (**b**) on PC-3 cells. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61), [69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

## 6 In Vivo Gene Silencing Activity

The in vivo efficacy of any siRNA/NV system is likely the phase of the entire NV-assisted RNAi process for which the development of standardized protocols is the most complex. Given the wide heterogenicity of genetic disorders, it is almost impossible to define a common roadmap for determining the actual efficacy of NV-mediated siRNA delivery and the corresponding gene silencing effect. Focusing on cancer , and even within a specific tumor, the number of variables to consider during any in vivo experimental campaign aimed at evaluating the efficacy of a therapeutic agent remains very large. Among these variables, however, reasonably the most important ones are represented by the tumor type, its location, the specific etiological agent from which cancer is initiated, and the corresponding diagnostic targets. Given such a broadness of subject, in this chapter final section we will still refer to our own experience and describe some of the main techniques and procedures adopted to verify the in vivo efficacy of siRNA/NV complexes in cancer—and in PC in particular. Yet, as it is the case of the methodologies discussed in the preceding sections, these techniques can easily be adopted and/or adapted for the in vivo characterization of NV-promoted RNAi aimed at treating different cancers/diseases. In our studies, PC-3 xenografted nude mouse models are generated to evaluate the in vivo NV-mediated silencing of *HSP27* as targeted by the corresponding siRNA. According to protocol, mice are sacrificed 1 week after siRNA/NV complex delivery via intratumoral injection, and the corresponding expression of Hsp27 mRNA and protein in the tumors is assessed by qRT-PCR and WB, respectively [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14), [23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23)]. It is important to remark here that the animal health must be constantly monitored, and that the institutional guidelines for the proper and ethical research animal use and sacrifice must be strictly followed in all in vivo experiment.

### 6.1 In Vivo Delivery and Gene Silencing

Our procedure implies measuring and recording mice tumor size and weight regularly during the entire siRNA treatment. After mice euthanasia, tumors are excised and readily subjected to gene silencing analysis. In order to be able to assess tumor targeting capability by ex vivo imaging, a fluorescently labeled siRNA must have been delivered [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)]. As a first example of the relevant results, Fig. [18](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig18) shows the ex-vivo imaging (Fig. [18a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig18)) and the relevant quantification (Fig. [18b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig18)) of the in vivo efficacy of *HSP27* gene silencing achieved upon delivery of the specific siRNA via the PAMAM-ACD-2 NV. Also, from the same figure the enhanced RNAi effect achieved upon NV decoration with the E16G6RGDK can be readily appreciated.

![485053_1_En_14_Fig18_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig18_HTML.png)

**Fig. 18** In vivo delivery of targeted siRNA/PAMAM-ACD-2/E16G6RGDK system. (**a**) Ex vivo imaging using Cy5-labeled siRNA/PAMAM-ACD-2/E16G6RGDK delivery system and (**b**) relative quantifications as mean florescence intensity. (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

Furthermore, Fig. [19](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig19) reveals the results from qRT-PCR (Subheading [5.1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec14)) and WB (Subheading [5.2](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec15)) analysis performed on the total Hsp27 mRNA and protein as extracted from the xenografted mice excised tumors. In agreement with the relevant in vitro data (Figs. [15](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig15) and [16](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig16)), these evidences confirm that a successful and potent *HSP27* silencing effect can be achieved in vivo upon specific siRNA delivery promoted by the PAMAM-CCD-G5 at mRNA (Fig. [19a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig19)) and protein level (Fig. [19b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig19)).

![485053_1_En_14_Fig19_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig19_HTML.png)

**Fig. 19** Effective and specific gene silencing of Hsp27 at both mRNA (**a**) and protein (**b**) levels promoted by siRNA/PAMAM-CCD-G5 complex. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61)] with permission of ACS)

### 6.2 Immunohistochemistry (IHC)

IHC is another microscopy-based technique that can be employed to visualize cellular proteins or other macromolecules in tissue samples upon staining with specific antibodies [[98](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR98)]. One of the IHC strong points is its intuitive output, which visually unveils the existence and localization of, e.g., a specific protein in the context of different cell types, biological states, and/or subcellular localization within complex tissues. For instance, in cancer therapeutics, high expression levels of the antigen Ki-67—a nuclear protein required for cellular proliferation—are associated with tumor aggression, vascular invasion, tumor metastasis, and poor response to chemotherapy [[99](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR99)–[102](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR102)]. As such, it is often used as a marker for the progression of cancer cells in tumor tissues [[102](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR102)]. An example of IHC images showing a strong inhibition of cancer cell proliferation after in vivo administration of siRNA in complexes with the PAMAM-CCD-G5 or the PAMAM-ACD-2 NVs is given in Fig. [20](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig20). As seen from the images, IHC of Ki-67 in tumors treated with the Hsp27 specific siRNA delivered by either of the two cationic NVs immediately reveals a substantially lower number of proliferative cells in the PC tissues with respect to the corresponding untreated controls.

![485053_1_En_14_Fig20_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig20_HTML.png)

**Fig. 20** Cancer cell proliferation in tumor tissue revealed by IHC using anti-Ki67 antibody as a cell proliferation marker. Immunohistochemical analysis for the Ki-67 expression when treated in vivo with siRNA/PAMAM-CCD-G5 (**a**) or with peptide coated or uncoated siRNA/PAMAM-ACD-2 (**b**) complexes. (Adapted from [[61](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR61), [69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

## 7 Toxicity

In spite of the wide applications of cationic dendrimers in drug delivery, their use in biomedical applications is still limited mainly because of their intrinsic toxicity. Indeed, the presence of a highly positively surface charge density on these NVs may result in cellular membrane disruption via nanohole formation, membrane thinning, and erosion [[103](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR103), [104](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR104)]. Therefore, in the development of any alternative, new cationic dendrimer-based NV , the determination of the relevant toxicity profile remains a mandatory step required for an eventual translational phase. In this respect, a comparison with the results obtained from a scrambled (aka non-silencing) siRNA, and the selection of cell lines not correlated with the studied pathology are also obligatory conditions.

### 7.1 MTT and Lactate Dehydrogenase (LDH) Assay

To investigate nonspecific toxicity, both the MMT assay (*see* Subheading [5.3](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Sec16)) and the LDH assay—which determines the membrane integrity by quantifying the lactate dehydrogenase released from cells into the extracellular medium [[105](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR105)]—can be routinely adopted. As illustrated in Fig. [21](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig21) for the delivery of *HSP27*-directed siRNA assisted by the siRNA/PAMAM-ACD-2 or the siRNA/PAMAM-ACD-2/E16G6RGDK NVs, the results from both MTT and LDH assays reveal that no metabolic toxicity or membrane damage is induced either in PC-3 cells (Fig. [21a, b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig21)) or in the disease-unrelated HEK293 control cells (Fig. [21c, d](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig21)) [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)].

![485053_1_En_14_Fig21_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig21_HTML.png)

**Fig. 21** Toxicity assessment of uncoated and coated siRNA/PAMAM-ACD-2 nanocomplex using the MTT assay and LDH assay for PC-3 prostate cancer cells (**a**, **b**) and HEK 293 human embryonic kidney cell (**c**, **d**). (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

### 7.2 Hemolysis Experiment

A potential toxic effect of nanoassemblies on red blood cells (RBCs), namely hemolysis, can be extremely dangerous during in vivo administration. Indeed, RBCs are among the first cells that come into contact with external materials in the blood system [[106](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR106), [107](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR107)]. Therefore, any eventual hemolytic toxicity—as marked by the release of hemoglobin from erythrocytes—must be carefully determined for every NV per se and in complex with its nucleic acid cargo. Briefly, washed RBCs suspensions must be incubated with different concentrations of the investigated nanoassembly; in parallel, positive and negative control experiments must be carried out. Specifically, a 1.0% solution of the surfactant TritonX-100 used as positive control causes a complete hemolysis of the sample, while pure phosphate buffered saline (PBS) is normally chosen as the negative control [[108](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR108)]. All samples are then centrifuged and the UV absorbance of the supernatant is measured at 540 nm to detect the release of hemoglobin from the eventually damaged cells. An example of the results from the hemolysis assay is given in Fig. [22](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig22), from which the enhancement in the protective effect against hemolysis achieved by, e.g., decorating the surface of the siRNA/PAMAM-ACD-2 complex with the E16G6RGDK targeting peptide with respect to the uncoated complex or to ACD nanovector alone is well evident. This result can be ascribed to a decrease in the overall positive surface charge characterizing the decorated nanoassembly [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)].

![485053_1_En_14_Fig22_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig22_HTML.png)

**Fig. 22** (**a**) Hemolysis assay of the siRNA/PAMAM-ACD-2 complexes and the siRNA/PAMAM-ACD-2/E16G6RGDK complexes in comparison with the pure ACD at different concentrations (0.50, 1.0, 2.5, 5.0, and 10 μM), which correspond to siRNA concentrations of 10, 20, 50, 100, and 200 nM. (**b**) Quantitative analysis of hemolysis determined by UV absorption at 540 nm. Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS

### 7.3 In Vivo Toxicity Evaluation

Finally, general in vivo toxicity assessments have to be performed to conclude the whole basic set of cationic NV characterization experiments. Generally, these include the administration of the siRNA/NV complex under development in healthy mice and the subsequent evaluation of any signs of discomfort, unusual behavior, or body weight change in comparison with a mice cohort treated with the PBS as a negative control. A reasonably complete in vivo toxicity profile of a NV/siRNA nanosystem can be obtained through the Luminex-detection technology-assisted quantification [[109](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR109)] of different induced inflammation and inflammatory factors—such as Interleukin 6 (IL-6) [[110](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR110)], Interferon-γ (INF-γ) [[111](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR111)], Tumor Necrosis Factor-α (TNF-α) [[112](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR112)], Granulocyte-macrophage colony-stimulating factor (GM-CSF) [[113](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR113)], Monocyte chemoattractant protein-1 (MCP-1) [[114](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR114)], Keratinocyte chemoattractant (KC) [[115](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR115)], and Interleukin 1β (IL-1β) [[116](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR116)]—and several major serum biochemistry parameters, including alanine transaminase (ALT) [[117](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR117)], aspartate transaminase (AST) [[118](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR118)], total bilirubin (TBIL) [[119](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR119)], urea [[120](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR120)], total protein (TP), alkaline phosphatase (ALP) [[121](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR121)], triacylglycerol (TG) [[122](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR122)], and total cholesterol (TCHO) [[123](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR123)]. A typical output from this analysis is reported in Fig. [23a, b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig23). In this example, healthy mice treated with either the E16G6RGDK targeted or the nontargeted siRNA/PAMAM-ACD-2 systems show no evidence of induced inflammation, whereas mice treated with lipopolysaccharide (LPS) as positive control exhibit significantly elevated levels of the abovementioned inflammatory factors (Fig. [23a](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig23)). Moreover, the major serum biochemistry parameters remain at normal levels at 24 h post-administration, indicating that the function of the main organs is not affected by the treatment with these siRNA/NV formulations (Fig. [23b](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig23)).

![485053_1_En_14_Fig23_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_14_Fig23_HTML.png)

**Fig. 23** In vivo toxicity assessment of the targeted siRNA/PAMAM-ACD-2/E16G6RGDK delivery system versus the nontargeted siRNA/PAMAM-ACD-2 system. (**a**) Cytokine response in mice determined by quantifying the serum levels of inflammation and inflammatory factors at 3 h post-injection. Lipopolysaccharide (LPS) was used as the positive control. (**b**) Major serum biochemistry parameters measured in mouse serum collected at 24 h post-injection. (**c**) Histopathological analysis of major organs from mice treated with siRNA/PAMAM-ACD-2 and siRNA/PAMAM-ACD-2/E16G6RGDK. Tissue samples were collected at 24 h post-administration. (Adapted from [[69](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR69)] with permission of ACS)

The corresponding histological analysis of the main organs collected at 24 h posttreatment (Fig. [23c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig23)) reveals that both siRNA/PAMAM-ACD-2 and siRNA/PAMAM-ACD-2/E16G6RGDK are devoid of acute toxicity since no significant pathological change can be observed in any of the tissue sections (Fig. [23c](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Fig23)).

## 8 Conclusion

In the last 10 years, the number of studies involving NVs as safe, efficient, and effective nanocarriers for siRNA delivery has increased exponentially. In particular, CCDs and ACDs PAMAM-based NVs bearing primary and positively charged amine groups are the optimal candidates for effectively forming nanoscale siRNA delivery systems. In this respect, our research group made significant progress in designing, synthesizing, characterizing, and testing both covalent and self-assembled nanovectors for siRNA delivery. We developed a hierarchical approach to control each phase of the experimental characterization on these NVs and the relevant complex with their nucleic cargos [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR14), [23](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#CR23)]. From our approach, an experimental guideline (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Tab1)) can be sketched in order to appropriately exploit specific experimental techniques for: (1) optimizing NVs in size and shape; (2) improving the stability of the siRNA/NV complex; (3) characterizing the uptake and the release of the nanocomplex at the cellular level; (4) testing in vitro and in vivo gene silencing activity of the siRNA/NV system; and, finally, (5) assessing its toxicity profile.

**Table 1**

Summary of the described experimental techniques and their corresponding outcomes for each characterization phase of NVs alone and siRNA/NV complexes

| Characterization phase                              | Experimental techniques                                      | Properties/outcomes                         |
| :-------------------------------------------------- | :----------------------------------------------------------- | :------------------------------------------ |
| (**I**) Pure NV characterization(I)(I)              | DLS                                                          | *D*H; *ζ*-potential                         |
| TEM                                                 | Imaging for size and morphology                              |                                             |
| ITC                                                 | CMC; self-assembly thermodynamics (Δ*G*mic, Δ*H*mic, *T*Δ*S*mic) |                                             |
| (**II**) siRNA/NV complex characterization(I)(I)(I) | DLS                                                          | *D*H; *ζ*-potential                         |
| TEM                                                 | Imaging for size and morphology                              |                                             |
| ITC                                                 | Binding thermodynamics (Δ*G*, Δ*H*, *T*Δ*S*)                 |                                             |
| Agarose gel                                         | Complex formation; stability against RNase degradation       |                                             |
| (**III**) Cellular uptake and release               | Flow cytometry                                               | Uptake level; targeting evaluation          |
| Confocal microscopy                                 | Uptake level and mechanism; release quantification and localization |                                             |
| (**IV**) In vitro silencing activity                | (qRT)-PCR                                                    | Inhibition of gene expression               |
| Western blot                                        | Inhibition of protein expression                             |                                             |
| MTT assay                                           | Cytotoxicity against target cell models                      |                                             |
| (**V**) In vivo silencing activity                  | Ex vivo imaging                                              | Tumor size reduction                        |
| (qRT)-PCR                                           | Inhibition of gene expression (in vivo)                      |                                             |
| Western blot                                        | Inhibition of protein expression (in vivo)                   |                                             |
| IHC                                                 | Inhibition of target cell proliferation                      |                                             |
| (**VI**) Toxicity assessment                        | MTT assay                                                    | Cytotoxicity against non-target cell models |
| LDH assay                                           | Membrane integrity of non-target cell models                 |                                             |
| Hemolysis experiment                                | Percentage of hemolytic effect                               |                                             |

In Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_14_Chapter.xhtml#Tab1), we sum up the techniques and their corresponding outcomes for each phase of the experimental characterization. Although this does not make an exhaustive and ultimate list, the reported information can be useful for the design and the development of safer and more effective NVs for siRNA delivery.