# Preparation of Lipid-Conjugated siRNA Oligonucleotides for Enhanced Gene Inhibition in Mammalian Cells 制备脂质偶联 siRNA 寡核苷酸以增强哺乳动物细胞的基因抑制能力 

Santiago Grijalvo, Sandra M. Ocampo, José Carlos Peralesand Ramon Eritja

## Abstract 摘要

Nucleic acid conjugates are promising drugs for treating gene-related diseases. Conjugating specific units like lipids, cell-penetrating peptides, polymers, antibodies, and aptamers either at the 3′- or 5′-termini of a siRNA duplex molecule has resulted in a plethora of siRNA bioconjugates with improved stabilities in bloodstream and better pharmacokinetic values than unmodified siRNAs. In this sense, lipid-siRNA conjugates have attracted a remarkable interest for their potential value in facilitating cellular uptake. In this chapter, we describe a series of protocols involving the synthesis of siRNA oligonucleotides carrying either neutral or cationic lipids at the 3′- and 5′-termini. The resulting lipid-siRNA conjugates are aimed to be used as exogenous effectors for inhibiting gene expression by RNA interference. A protocol for the formulation of lipid siRNA using sonication in the presence of serum is described yielding interesting transfection properties for cell culture without the use of transfecting agents.

<span style=color:blue>核酸偶联物有望成为治疗基因相关疾病的药物。将脂质、细胞穿透肽、聚合物、抗体和适体等特异性单元与 siRNA 双链分子的 3' 或 5' 末端结合，产生大量 siRNA 生物偶联物，这些偶联物具有比未修饰 siRNA 更高的血液稳定性和更好的药代动力学参数。从这个意义上说，脂质-siRNA 偶联物因其促进细胞摄取的潜在价值而备受关注。本章描述了一系列涉及合成携带中性或阳离子脂质的 3' 和 5' 末端 siRNA 寡核苷酸的方案。所得的脂质-siRNA 缀合物旨在作为外源效应物，通过 RNA 干扰抑制基因表达。此外，本章还介绍了在含血清的环境下，通过超声处理制备脂质siRNA的方法，这种方法无需使用转染试剂就能为细胞培养提供有效的转染特性。</span>

## 1 Introduction 引言

Nucleic acids represent a class of biomolecules that contain a great number of biological properties. Since Zamecknik and Stephenson reported in 1978 the use of a 13-mer antisense oligonucleotide as a tool for blocking the replication process of the RNA Rous sarcoma virus, new horizons and novel strategies have been opened up in the development of oligonucleotides as promising drug therapeutics. In fact, the first DNA oligonucleotide (Fomivirsen) approved by the Food and Drug Administration (FDA) for clinical use was effective in 1998. Since then it has come a long way making significant strides in the field of nucleic acid therapeutics with the approval of eight more antisense oligonucleotides (e.g., Pegaptanib, Mipomersen, Eteplirsen, Defibrotide, Nusinersen, and Inotersen) and two RNA interference-based therapeutics (e.g., Patisiran and Givosiran), the latter being accepted for human use in 2018 and 2019, respectively. This success has allowed nucleic acids to be launched to several advanced clinical trials and becoming a promising alternative to traditional therapy when using small molecule drugs to combat gene-related diseases.

<span style=color:blue>核酸是一类具有众多生物学特性的生物分子。自1978年Zamecknik和Stephenson首次报道利用13聚反义寡核苷酸阻断RNA Rous肉瘤病毒的复制以来，寡核苷酸作为药物治疗工具的开发开辟了新的前景和策略。实际上，第一个获得美国食品药品监督管理局（FDA）批准用于临床的DNA寡核苷酸药物Fomivirsen于1998年投入使用。自那以来，核酸疗法领域取得了显著进展，八种反义寡核苷酸（如Pegaptanib、Mipomersen、Eteplirsen、Defibrotide、Nusinersen和Inotersen）和两种基于RNA干扰的治疗药物（如Patisiran和Givosiran）相继获批，后者分别在2018年和2019年获准上市。这一成就使得核酸疗法进入多个高级临床试验阶段，并成为小分子药物治疗基因相关疾病的传统疗法的有力替代方案。</span>

RNA interference (RNAi) is a powerful process, discovered by Fire and Mello, in which a 21–25 base pair double-stranded small interference RNA (siRNA) is able to trigger the RNA-induced silencing complex (RISC) and interact specifically with a messenger RNA (mRNA). In this regard, this process degrades such mRNA inhibiting translation step by blocking protein production. Interestingly, this mechanism of action is catalytic making RNAi to be superior and more efficient in inhibiting specific target proteins. Despite this silencing activity, siRNA-based therapeutics must cope with many challenges including stability in bloodstream and low transfection efficiencies, among others. While formulation strategies have been developed to avoid degradation and therefore enhance cellular internalization, decoration of such nano-formulations with targeting ligands has also improved effectiveness and selectivity of siRNA molecules in vivo. Some comprehensive reviews have been recently published on this topic.

<span style=color:blue>RNA干扰（RNAi）是由Fire和Mello发现的一种强效机制，通过21-25碱基对的双链小干扰RNA（siRNA）触发RNA诱导的沉默复合物（RISC），并与信使RNA（mRNA）特异性结合，从而降解mRNA，阻断蛋白质的翻译过程。这种催化性机制使得RNAi在抑制特定目标蛋白质方面更为高效。尽管如此，siRNA疗法仍面临许多挑战，如在血液中的稳定性和低转染效率等。为了解决这些问题，已经开发了多种配方策略以防止降解并提高细胞内化。此外，通过在纳米配方上装饰靶向配体，进一步提高了siRNA分子的有效性和选择性。最近，关于这一主题已有多篇综合性综述发表。</span>

It is well known that also modifying chemically certain positions of natural nitrogenous bases and sugar rings has remarkably improved stability without compromising original siRNA activities. On the other hand, conjugating either 3′- or 5′-end positions with bioactive molecules like lipids, cell-penetrating peptides, polymers, and/or antibodies has enabled the preparation of more stable siRNA conjugates when compared to their siRNA complex counterparts. In this regard, the use of chemical strategies has resulted in synthesizing siRNA bioconjugates with improved stabilities, potency, targetability as well as exhibiting better siRNA fate when administered in vivo. The development of various chemical strategies to modify siRNA backbone with lipids, cell-penetrating peptides, aptamers, and polymers has been reviewed in many revision articles.

<span style=color:blue>众所周知，通过化学修饰天然氮碱基和糖环的特定位置可以显著提高siRNA的稳定性，同时不会影响其原有活性。另一方面，将生物活性分子如脂质、细胞穿透肽、高分子和/或抗体连接到siRNA的3′或5′末端，能够制备出比传统siRNA复合物更稳定的siRNA共轭物。在这方面，化学策略的应用使得合成的siRNA生物共轭物具有更高的稳定性、效力和靶向性，并在体内给药时表现出更好的效果。许多综述文章已经回顾了通过脂质、细胞穿透肽、适配体和高分子修饰siRNA骨架的各种化学策略的发展。</span>

The presence of hydrophobic pendent groups has made RNA oligonucleotide chains to increase in lipophilicity and thus turn into more drug-like molecules to induce a specific activity in vitro and in vivo. Several chemical strategies have been successfully developed to modify siRNA backbones with lipid moieties either at 5′- or 3′-termini of the passenger strand including both stable and cleavable linkages. For instance, a direct conjugation involving a lipid residue and siRNA was reported by Kubo et al. in 2013. Such chemical combination required a series of siRNAs which were modified properly with an amino modifier and a *N*-hydroxysuccinimide ester derivative that was attached to the corresponding lipid chains. This protocol allowed the authors to introduce covalently hydrophobic residues into a series of siRNAs at the 5′-end in the presence of *N*,*N*-diisopropylethylamine (DIEA) and a mixture of isopropanol:water (1:1).

Introducing lipid moieties at the 3′-termini of siRNAs requires additional synthetic efforts based on derivatizing Controlled Pore Glass (CPG) solid supports with small molecule linkers. These units containing hydrophobic residues have been further modified with specific acid-labile protecting groups like trityl (Tr), 4-monomethoxy trityl (MMTr), or 4,4′-dimethoxy trityl (DMTr) giving rise to lipid-modified CPG solid supports that have been used in conjunction with automatic oligonucleotide synthesizers. In this regard, many examples have been described for the chemical conjugations of cholesterol and other lipids (e.g., α-tocopherol, lithocolic-oleyl, docosanoic acid, docosahexaenoic acid, and eicosapentaenoic acid) with siRNA oligonucleotides showing improved silencing activities and specific accumulations in liver but also other tissues like skeletal muscle and heart when injected intravenously (Fig. 1). While internalization mechanisms of some siRNA conjugates remain unclear, tissue uptake and cellular internalization efficacies are driven not only by the hydrophobic nature of the lipid pendent groups but also the effective binding of such conjugates to serum proteins such as high-density and low-density lipoproteins (HDL and LDL, respectively) as well as albumin.

![../images/485053_1_En_8_Chapter/485053_1_En_8_Fig1_HTML.png](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig1_HTML.png)

**Fig. 1** Scheme of some of the chemical structures of lipids that have been incorporated in siRNA

In an effort to find additional strategies addressed to enhance the efficiency of siRNAs in transfection processes, our research group has studied the effect of several lipid modifications introduced either at the 3′- or 5′-termini of a siRNA passenger strand taking TNF-α as a target protein model (Fig. 2). TNF-α was selected because overexpression of this gene is found associated to many diseases such as cancer, autoimmune diseases: rheumatoid arthritis, ankylosing spondylitis, Crohn’s disease, psoriasis, refractory asthma. In order to connect the lipid moiety to the RNA molecules, we selected glycerol as a small linker to prepare a series of siRNA conjugates containing lipid pendent groups in the form of ether lipids both neutral and cationic at the 3′-end. On the other hand, other neutral and cationic lipids were also used to modify siRNA passenger strand at the 5′-termini by using the well-known phosphoramidite chemistry strategy.

![485053_1_En_8_Fig2_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig2_HTML.png)

**Fig. 2** Schematic representation of some glycerol-based lipids described for the preparation of lipid-siRNAs modifying the RNA passenger strand

Herein, we describe specific experimental protocols that have been carried out in the field of siRNA delivery with the aim to increase both its potency and transfection efficiency. Particularly, we describe a protocol to synthesize a series of siRNA conjugates carrying neutral and cationic lipids at the 3′- and 5′-termini modifying chemically the siRNA guide strand (Fig. 2 and Table 1). In addition, a formulation method used for increasing the binding capacity of unmodified and chemically modified siRNAs through a process namely “pre-binding” is described. This protocol that we named as “*f-siRNA*” protocol is based on a short sonication of lipid-siRNA conjugates in the presence of serum with the aim to facilitate their binding to serum proteins such as albumin, HDL, and LDL. The ultrasound treatment carried out before adding the siRNA conjugates to cell culture showed a clear enhancement of cellular uptake and therefore resulted in an enhancement of the silencing activities promoted by such siRNA conjugates in vitro when compared with non-ultrasound-treated oligonucleotides.

**Table 1** Lipid-siRNA conjugates carrying glycerol-based long linear alkyl chains. 

Structures for Chol, *lipid*_*C14*, *lipid*_*C18,* and *lipid*_*C14N* are shown in Fig. 2. Underlined residues indicate the position of 2′-*O*-methyl-RNA residues

| Code | Modification                 | Sequence                                                     |
| :--- | :--------------------------- | :----------------------------------------------------------- |
| w.m  | Unmodified anti-TNFα         | 5′-GAGGCUGAGACAUAGGCACTT-3′ (*guide strand*)5′-GUGCCUAUGUCUCAGCCUCTT-3′ (*passenger strand*) |
| Chol | Sense-3′-cholesterol (Chol)  | 5′-GUGCCUAUGUCUCAGCCUCTT-***Chol***-3′ (*passenger strand*)  |
| 106  | Sense-3′-*Lipid_C14*         | 5′-GUGCCUAUGUCUCAGCCUCTT-***Lipid_C14***-3′ (*passenger strand*) |
| 107  | Sense-3′-*Lipid_C18*         | 5′-GUGCCUAUGUCUCAGCCUCTT-***Lipid_C18***-3′ (*passenger strand*) |
| 110  | Sense-3′-*Lipid_C14* and OMe | 5′-GUGC**C**UAUG**UC**U**C**AG**C**C**U**CTT-***Lipid_C14***-3′ (*passenger strand*) |
| 111  | *Sense-5′-Lipid_C14N*        | 5′-***C14N***-GUGCCUAUGUCUCAGCCUCTT-3′ (*passenger strand*)  |
| SCR  | Unmodified scrambled         | 5′-CAGUCGCGUUUGCGACUGGTT-3′ (*guide strand*)5′-CCAGUCGCAAACGCGACUGTT-3′ (*passenger strand*) |

## 2 Materials

Use deionized pure water with a resistivity of 18.2 MΩ to dissolve both RNA strands (passenger and guide) and prepare buffer solutions (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). Reagents were used as received without being further purified.

### 2.1 RNA Oligonucleotide Synthesis

Perform the protocols described below in a DNA synthesizer (Applied Biosystems 3400) using two different scales: LV200 and 1 μmol (*see* **Note** [**2**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). Use solvents and ancillary reagents with DNA synthesis grade (*see* **Note** [**3**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)).

1. A computer equipped with the 3400 DNA Synthesizer software. 
2. A DNA synthesizer (e.g., Applied Biosystems 3400). 
3. Unmodified DNA controlled-Pore Glass (CPG) solid support (3′-dT-CPG) (*see* **Note** [**4**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
4. Unmodified RNA CPG solid supports (Bz-A-RNA, Ac-C-RNA, Ac-G-RNA and U-RNA) (*see* **Note** [**5**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
5. RNA 2-cyanoethyl (CE) phosphoramidites (Bz-A-CE, Ac-C-CE, dmf-G-CE and U-CE phosphoramidites) (*see* **Note** [**5**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
6. RNA phosphoramidites modified at their 2′-position of the sugar with OMe (2′-OMe-A-CE, 2′-OMe-Ac-C-CE, 2′-OMe-dmf-G-CE and 2′-OMe-U-CE phosphoramidites) (*see* **Note** [**6**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
7. Anhydrous acetonitrile. 
8. 3% Trichloroacetic acid (TCA) in dichloromethane. 
9. Anhydrous dichloromethane. 
10. 0.4 M 1*H*-tetrazol in acetonitrile. 
11. Capping A: Acetic anhydride/pyridine/tetrahydrofurane (1:1:8). 
12. Capping B: 10% *N*-methylimidazole in tetrahydrofurane. 
13. 0.01 M Iodine in tetrahydrofurane/pyridine/water (7:2:1) (*see* **Note** [**7**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
14. Concentrated ammonia (32%) in water. 
15. Triethylamine trihydrofluoride, 98% (TREAT-HF). 
16. Triethylamine. 
17. *N*-Methylpyrrolidone, 97%. 
18. Isopropoxytrimethylsilane, 98%. 
19. Annealing buffer: 10 mM Tris-Base [Tris(hydroxymethyl)aminomethane], 50 mM NaCl, 1 mM EDTA (ethylenediaminetetraacetic acid), pH 8.0. 
20. 3 M Sodium acetate solution, pH 5.2. 
21. Ethanol. 
22. A centrifuge 5424 R instrument. 
23. Argon or nitrogen sources. 
24. 2 mL Glass screw vials. 
25. Dry bath tube heating block. 
26. Speed Vac evaporator. 

### 2.2 High-Performance Liquid Chromatography (HPLC)

Detach RNA single strand oligonucleotides from CPG solid supports once RNA synthesis has been completed (*see* Subheading [3.1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec11)). Analyze and purify by reverse-phase HPLC (RP-HPLC) in order to obtain the corresponding RNA oligonucleotide strands (guide or passenger) with high purity.

1. Analytical and/or semi-preparative RP-HPLCs are both equipped with a Waters 2695 Separation Module and a Waters 2998 Photodiode Array Detector. 
2. A computer equipped with Empower™ 3 chromatography data software. 
3. A HPLC reverse-phase column C18, with a size pore of 2.5 μm, and 4.6 × 50 mm column (*see* **Note** [**8**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
4. Gas-tight syringe.
5. 2 mL Conical plastic tubes (*see* **Note** [**9**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
6. 1 M Aqueous triethylammonium acetate (TEAA): acetic acid, triethylamine, water, pH 7.01 (stock solution). 
7. Buffered solution A (1 L): 5% acetonitrile, 0.1 M aqueous TEAA, pH 7.01. 
8. Buffered solution B (1 L): 70% acetonitrile, 0.1 M aqueous TEEA, pH 7.01. 

### 2.3 Desalting RNA Oligonucleotide Solutions

1. Disposable NAP DNA/RNA size-exclusion purification columns (NAP-5 or NAP-10 columns) (*see* **Note** [**10**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
2. Sephadex G-25. 
3. RNAse-free water (DEPC-water) (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
4. Eyedropper. 
5. 2 mL Conical plastic tubes (*see* **Note** [**9**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 

### 2.4 Ultraviolet-Visible (UV) Spectroscopy

1. A computer equipped with the Spectra Manager™ II software. 
2. A Jasco V-650 UV-VIS spectrophotometer equipped with a thermoregulated cell holder. 
3. A 1 mL-UV quartz Hellma absorption cuvette with PTFE stoppers. 
4. RNAse-free water (DEPC-water) (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
5. Eyedropper. 
6. 2 mL Conical plastic tubes (*see* **Note** [**9**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 

### 2.5 Matrix-Assisted Laser Desorption/Ionization Time-of-Flight (MALDI-TOF) Mass Spectrometry

Identify and characterize the correct mass of RNA oligonucleotides carrying lipid pendent groups using a MALDI-TOF mass spectrometry .

1. *Perseptive* Voyager-DE™RP spectrometer (Applied Biosystems) in negative mode equipped with nitrogen laser at 337 nm with a 3 ns-pulse. 
2. A computer equipped with Data Explorer™ software. 
3. 2,4,6-Trihydroxyacetophenone (THAP): 10 mg/mL in a mixture of acetonitrile:water (1:1). 
4. Ammonium citrate (CA): 50 mg/mL aqueous solution (*see* **Note** [**11**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
5. RNAse-free water (DEPC-water) (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 

### 2.6 Cell Culture Models

1. Immortal human cervical cancer HeLa cells or 4T1 murine breast cancer cells. 
2. Dulbecco’s Modified Eagle Medium (DMEM) with Glutamax supplement (500 mL). 
3. Opti-MEM reduced serum medium (500 mL). 
4. 1× Phosphate-buffered saline solution (PBS) (500 mL). 
5. Fetal bovine serum (FBS) (500 mL). 
6. Penicillin and streptomycin (100 U/mL and 100 mg/mL, respectively). 
7. Lipofectamine™ 2000 transfection reagent (0.75 mL, ThermoFischer Scientific). 

### 2.7 Enzyme-Linked Immunosorbent Assay (ELISA)

Detect and quantify the presence of ligands or antigens by carrying out the ELISA assay. To do so, use the Murin TNF-α Instant Elisa (Bender MedSystems) instead of the conventional one to detect quantitatively TNF-α from the cell culture supernatant following the manufacturer’s instructions.

1. A pre-coated 96-well microplate. 
2. A biotinylated detection antibody.
3. A SAV-HRP conjugate, generally an antibody or streptavidin. 
4. Diluent and wash buffers. 
5. A substrate solution. 
6. Lysis buffer 10×.

## 3 Methods

### 3.1 Chemical Synthesis of Lipid-siRNA

#### 3.1.1 Preparation of 3′- and 5′-Lipid RNA Oligonucleotide Conjugates

Glycerol-based CPG solid supports as well as modified neutral and cationic lipid phosphoramidites were previously synthesized according to well-established experimental procedures [[13](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR13), [14](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR14)].

1. Weigh 20–30 mg of commercially available CPG modified with cholesterol (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1); Chol) and glycerol-based CPG solid supports (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1); 106, 107, 110, 111, and SCR). Place them in a DNA synthesizer (*see* **Note** [**12**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
2. Dissolve the corresponding lipid phosphoramidite (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1); **8**) in an appropriate anhydrous solvent like acetonitrile (ACN) to give a final concentration of 0.1 M (*see* **Note** [**13**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
3. The resultant lipid phosphoramidite solutions must be finally integrated into the automated synthesizer. 
4. Program the sequences (both guide and passenger strands) displayed on Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1) in the DNA/RNA synthesizer. 
5. Remove the modified CPG solid supports (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1)) carrying the desired oligonucleotides and the selected lipids once the automated synthesis has ended up (*see* **Note** [**14**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)).
6. Transfer the solid supports to a 2 mL-screw glass vial. Add 1 mL of a mixture of concentrated ammonia (32%) in ethanol (3:1) and heat the suspension up at 55 °C for 1 h. 
7. Filter the CPG solid supports off and wash with additional ethanol in autoclaved conical tubes. 
8. Evaporate the mixture of solvents (water and ethanol) in a Speed Vac evaporator to dryness. 
9. Dissolve the residue containing the RNA oligonucleotide conjugates with 0.15 mL solution containing triethylamine trihydrofluoride, triethylamine, and *N*-methylpyrrolidone (4:3:6) for 2.5 h at 65 °C in order to remove both the silyl (TBDMS) and 3,3′-dimethoxytrytil (DMTr) protecting groups. 
10. Add isopropoxytrimethylsilane (0.3 mL) to stop the deprotection reaction and diethyl ether (0.75 mL). 
11. Vortex the final mixture solutions and cool down overnight at 4 °C to facilitate the precipitation of the expected DMT*off*-RNA oligonucleotide conjugates (*see* **Note** [**15**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
12. Centrifuge conical tubes containing RNA oligonucleotide conjugates at 14,086 × *g* at 4 °C for 5 min. 
13. Wash the corresponding pellets with diethyl ether and centrifuge again using the same conditions as described above. 
14. Dry the RNA pellets carefully using a weak nitrogen stream . 

#### 3.1.2 Purification of RNA Oligonucleotide Conjugates by HPLC

1. Dissolve the RNA pellets obtained in Subheading [3.1.1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec12) in RNAse-free miliQ-water or DEPC-treated water. 
2. Use the appropriate desalting columns (Sephadex NAP-5 or NAP-10 columns) following the manufacturer’s instructions. 
3. Use “Solvent A” and “Solvent B” as buffered solutions (*see* Subheading [2](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec2)) and placed them in labeled bottles. 
4. Analyze the corresponding RNA oligonucleotide conjugates. Run a 10-min linear gradient starting at 0% to 35% B and a flow rate of 1 mL/min (*see* **Note** [**16**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)).
5. Program a specific method and use a flow rate of 3 mL/min and 20-min linear gradient from 15% to 80% B maintaining 5 additional minutes to 80% B to purify the RNA oligonucleotide conjugates (*see* **Note** [**16**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
6. Load a syringe with 200 μL crude RNA oligonucleotide solutions (*see* **Note** [**17**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
7. Collect pure RNA oligonucleotide samples in conical tubes and use a Speed Vac evaporator to concentrate the samples to dryness. 
8. Characterize the collected fractions by analytical HPLC and mass spectrometry (MALDI-TOF). Use a UV-spectrophotometer to calculate the final concentration of the pure RNA oligonucleotide strand. 
9. An example of a HPLC chromatogram and a MALDI-TOF spectrum of a modified lipid RNA passenger strand is shown in Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig3). 

![../images/485053_1_En_8_Chapter/485053_1_En_8_Fig3_HTML.png](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig3_HTML.png)

**Fig. 3** (**a**) HPLC purification of sense-3′-*lipid*_*C14* and OMe (110). (**b**) Mass spectrum (MALDI-TOF) of the main peak (expected 6993; found 7013 [M + Na+])

#### 3.1.3 Annealing and Precipitation of siRNA Duplexes

1. Take those lyophilized RNA strands modified with the appropriate lipid pendent groups and RNA guide strands.
2. Dissolve both strands in annealing buffer (100 μL).
3. Mix equimolar amounts of each RNA strands.
4. Heat the mixture of both strands up at 93 °C and cool down slowly at 25 °C for 5 h. 
5. Add 3 M AcONa solution (10 μL) and vortex the final solution. 
6. Add 3 vol. of EtOH and vortex again. 
7. Store the final solution at −18 °C overnight to trigger the precipitation of the lipid-siRNA conjugates.
8. Isolate and dissolve the precipitated siRNA in an appropriate buffer and quantify it by UV spectroscopy at 260 nm wavelength. 

### 3.2 Evaluation of the TNF-α Inhibition Properties

#### 3.2.1 In Vitro Gene Silencing Studies

Use these general protocols with the aim to transfect the corresponding unmodified and modified lipid-siRNA conjugates in two specific adherent cell culture lines (e.g., HeLa and 4T1). Detailed protocols are described below.

1. Use HeLa and 4T1 cell lines for in vitro gene silencing studies to inhibit TNF-α gene expression (*see* **Note** [**18**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)).
2. Transfect a plasmid expressing such gene (pCAm TNF-α; 250 ng) using a commercially available cationic lipid, namely Lipofectamine™ 2000 when working with HeLa cell lines following the manufacturer’s instructions (*see* **Note** [**18**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)).
3. Incubate both unmodified and siRNA bioconjugate duplexes (100 nM) displayed on Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1) with fetal calf serum (FCS) previously (*see* the “Pre-binding Strategy” section for further details) (Fig. [4](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig4)).
4. Transfect either HeLa or 4T1 cells with the corresponding siRNA complexes.
5. Use the same siRNA duplexes and transfect them using the cell lines described before without being incubated with fetal calf serum (FCS).
6. Transfection processes in **steps 4** and **5** must be performed in the absence of Lipofectamine™ 2000. 

![485053_1_En_8_Fig4_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig4_HTML.png)

**Fig. 4** Schematic representation of the sonication method (f-siRNA)

##### Pre-binding Strategy

1. Culture and grow exponentially HeLa cells under standard conditions (37 °C, 5% CO2) in DMEM as a suitable medium supplemented with 10% FBS and antibiotics (*see* **Note** [**19**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
2. Remove the growing media and use 500 μL of fresh DMEM supplemented with 10% FBS without antibiotics to perform the transfection experiments. 
3. Use a 40–60% confluence to perform the transfection experiments. 
4. Use 250 ng of a plasmid expressing murine TNF-α (pCAm TNF-α) and mix with Lipofectamine™ 2000 in Opti-MEM (300 μL). 
5. Incubate the resulting liposomes for 15 min at room temperature. 
6. Add 100 μL of the preformed liposome solution in three wells to get a final volume of 600 μL. 
7. Incubate HeLa cells for 4 h under standard conditions (37 °C, 5% CO2) (*see* **Note** [**20**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
8. Wash thoroughly with 1× PBS in order to remove the lipid-based transfecting agent. 
9. Mix those siRNA duplexes (100 nM) displayed on Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Tab1) with FCS in the ratio of 0.66 μg siRNA/25 μL FCS. 
10. Place siRNA/serum complexes on a sonication bath for 5 min (*see* **Note** [**21**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
11. Add the resulting siRNA/FCS complexes (“*f-siRNA*”) to HeLa cultured cells in the presence of Opti-MEM medium and in the absence of a commercially available cationic lipid. 
12. Incubate HeLa cells containing *f-siRNA* for 4 h under standard conditions (37 °C, 5% CO2). 
13. Remove media and add fresh DMEM supplemented with 10% FBS after 4 h-incubation. 
14. Incubate HeLa cells for 48 h under standard conditions (37 °C, 5% CO2). 
15. Analyze the amount of TNF-α produced by HeLa cells by the enzyme-linked immunosorbent assay (ELISA) according to manufacturer’s instructions (*see* **Note** [**22**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
16. Results are shown in Figs. [5](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig5) and [6](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig6).

![485053_1_En_8_Fig5_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig5_HTML.png)

**Fig. 5** Efficiency of “f-siRNA” method for silencing TNFα in 4T1 cells. Left: without sonication. Right: with sonication. 4T1 cells were transfected with 100 nM of each “f-siRNA,” after 24 h levels of TNFα were measured by ELISA. The data represent the mean ± SE, *n* = 3 and are compared with Scrambled sequence. **p* < 0.05, ***p* < 0.01, ****p* < 0.001. ANOVA, Bonferroni post-test. w.m unmodified anti-TNFα siRNA. Chol: sense 3′-Cholesterol, 106: sense 3′*lipid*_*C14*, 107: sense 3′ *lipid*_*C18*, 110: sense 3′ *lipid*_*C14*-OMe, 111: sense 5′ *lipid*_*C14N*, Scr: Scrambled

![485053_1_En_8_Fig6_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_8_Fig6_HTML.png)

**Fig. 6** (**a**) Silencing of TNFα in HeLa cells. Gray (control), HeLa cells were transfected with 250 ng of TNFα plasmid pCAm using Lipofectamine™ 2000 as a vector transfection, 1 h after 100 nM of each of the siRNAs were transfected directly on the cell medium supplemented with serum. After 48 h, levels of TNFα were measured by ELISA. Black (f-siRNA), HeLa cells were transfected with 250 ng of plasmid pCAm TNFα using Lipofectamine™ 2000 as a vector, then were transfected 1 h later with 100 nM of each “f-siRNAs.” After 48 h levels of TNFα were measured by ELISA. (**b**) Silencing of TNFα in 4T1 cells. Gray (control), 4T1 cells were transfected with 100 nM of each siRNAs directly on the cell medium supplemented with serum. After 24 h, levels of TNFα were measured by ELISA. Black (f-siRNA), 4T1 cells were transfected with 100 nM of each “f-siRNA”, after 24 h levels of TNFα were measured by ELISA. The data represent the mean ± SE, *n* = 3 and are compared with Scrambled sequence **p* < 0.05. Statistical analysis was performed by ANOVA, Bonferroni post-test. siRNAs: siTNF, unmodified siRNA. siTNF-Chol: sense 3′-Cholesterol, siTNF-C14: sense 3′lipid *C14* (106), Scr: Scrambled

In the case of using 4T1 breast cancer cells use only **steps 9**–**14** (without transfection of the pCAm plasmid).

##### Non-pre-binding Strategy

1. Perform the same experiments using same siRNA duplex amounts but without sonicating them in the presence of FCS. 
2. Add siRNA duplexes (100 nM) to HeLa cultured cells in the absence of a commercially available cationic lipid and incubate cells and siRNA duplexes for 48 h under standard conditions (37 °C, 5% CO2). 
3. Analyze the amount of TNF-α produced by HeLa cells by the enzyme-linked immunosorbent assay (ELISA) according to the manufacturer’s instructions . 
4. Results are shown in Figs. [5](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig5) and [6](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig6). 

## 4 Notes

1. One should be extremely careful when working with RNA oligonucleotides. In this regard, glassware and plasticware should be autoclaved before putting in contact with RNA. In addition, DEPC-treated water or nuclease-free water is required to avoid the degradation of both RNA oligonucleotides and siRNA bioconjugates. To prepare DEPC-treated water, 1 mL of diethylpyrocarbonate (DEPC) is added to 1 L of water. The resultant solution is incubated at room temperature overnight and finally autoclaved. 
2. This is a solid-phase automated process in which building blocks (DNA or RNA phosphoramidites) are coupled to a growing oligonucleotide in an orderly manner leading to a specific sequence that has been previously programmed. The average yield for the coupling of RNA monomers is usually around 97–98%. Several building blocks derived from the phosphoramidite method (e.g., protected 2′-deoxynucleosides, ribonucleosides, locked nucleic acids and bridge nucleic acids) can be used in an automated oligonucleotide synthesis. 
3. Ancillary reagents used during the RNA synthesis are the following: in DCM (detritylation step); 0.4 M 1*H*-tetrazol in ACN (activation step); acetic anhydride/pyridine/tetrahydrofurane (THF) (1:1:8) (capping A); 10% *N*-methylimidazole in THF (capping B) and 0.01 M iodine in THF/pyridine/water (7:2:1) (oxidation step). This oxidizing agent readily oxidizes the phosphorous (III) to phosphorous (V). These ancillary reagents are available from Merck Sigma-Aldrich (Madrid, Spain), Link Technologies (Scotland, UK), Glen Research (Sterling, VI, USA), or Applied Biosystems (Foster City, CA, USA) among many others. 
4. Unmodified siRNA duplexes usually have their 3′-end passenger and guide strands modified with two thymidines (dT) with the aim to protect siRNAs from degradation. However, siRNAs without bearing these dTs at the ends have been also prepared and used in cell culture experiments. 
5. Common protecting groups are the following: *Bz* is a benzoyl group; *Ac* is an acetyl and *dmf* is a dimethylformamidine protecting groups. The 2′-OH protecting group used for natural RNA monomers is usually the *tert*-butyldimethylsilyl (TBDMS) protecting group although other silyl derivatives have been also reported like TOM (triisopropylsilyloxymethyl) [[21](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR21)]. 
6. Several companies can provide not only regular CPG solid supports and natural RNA phosphoramidites as described before, but also minor 2′-OMe-RNA phosphoramidites like Glen Research (Sterling, VI, USA) to afford RNA oligonucleotides with improved stabilities. 
7. In addition to obtaining phosphate linkages in the presence of iodine, water, and a weak base, other oxidizing reagents like *tert*-butyl hydroperoxide [[22](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR22)] and (1*S*)-(+)-(10-camphorsulfonyl)-oxaziridine (CSO) [[23](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR23)] can be also used. On the other hand, sulfurizing reagents can be integrated into automatized synthesis in order to replace one of the phosphate’s oxygens in the oligonucleotide chain by a sulfur atom and therefore obtaining RNA oligonucleotide derivatives containing phosphorothioate linkages. To do so, *N*,*N*,*N*,*N*-tetraethylthiuram disulfide (TETD) [[24](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR24)], Beaucage reagent [[25](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR25)] or 3-(dimethylaminomethylidene)amino-3*H*-1,2,4-dithiazole-3-thione (DDTT) [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#CR26)] have been reported giving rise to the expected phosphorothioate linkages. It is worth mentioning that using DDTT affords better sulfurization kinetics as well as stability in solution when comparing to other sulfurizing reagents described above. 
8. Reverse and anion-exchange HPLC columns can be chosen to analyze and purify oligonucleotides (e.g., XBridge™ C18, DNAPac RP LC columns, DNASwift™ SAX-1S, Nucleosil 120-10 C18 (250 × 4 mm), among others). 
9. Polypropylene or polystyrene conical tubes should be previously autoclaved (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Sec19)). 
10. NAP DNA/RNA columns (GE Healthcare) are available in three sizes: NAP-5 (0.5 mL), NAP-10 (1.0 mL), and NAP-25 (2.5 mL). The number in brackets is the dead volume of the columns. 
11. Other common matrices used in MALDI-TOF mass spectrometry are: sinapinic acid (SA), α-cyano-4-hydroxycinnamic acid (ACH), 2,5-dihydroxybenzoic acid (DHB), or 3-hydroxypicolinic acid (3HPA), among others. 
12. The required amount of CPG solid support for oligonucleotide synthesis will rely on DMTr-functionalization by calculating the resin loading. Interestingly, CPG solid supports modified with hydrophobic residues like cholesterol and palmitate can be found commercially available from Link Technologies (Scotland, UK) or Glen Research (Sterling, VI, USA). 
13. Some lipid phosphoramidites can be also dissolved in a mixture of ACN and dichloromethane (8:2) to obtain a 0.1 M concentration. In addition, many elaborated lipid phosphoramidites like 5′-tocopherol, palmitate, cholesterol, octyl-tocopherol, spermine, or stearyl can be also found commercially available from Link Technologies (Scotland, UK), Glen Research (Sterling, VI, USA), Berry & Associates (Dexter, MI, USA), ChemGenes (Wilmington, MA, USA), Metkinen (Finland), or TriLink Technologies (San Diego, CA, USA), among many others. 
14. Unmodified anti-TNF-α and scrambled RNA sequences were obtained following the same experimental protocols used for lipid-RNA oligonucleotide conjugates. On the other hand, such unmodified and scrambled RNA sequences can be also obtained from commercial sources (ATD Bio, Eurogentec, Merck Sigma-Aldrich, among many others). 
15. The use of tetrabutylammonium fluoride (TBAF) instead of triethylamine trihydrofluoride does not remove the last 5′-DMTr protecting group of the RNA oligonucleotide sequence. In this sense, an additional treatment with 80% AcOH is required to remove the final DMTr protecting group. 
16. The gradient program may vary and depend on the modification chemical nature introduced in the RNA oligonucleotide. 
17. Try to avoid saturating HPLC column by getting UV-absorbances greater than 1.0. 
18. HeLa cells do not express naturally the murine TNF-α gene; therefore a plasmid expressing such gene (pCAm TNF-α; 250) must be previously transfected in the presence of a commercially available cationic lipid, namely Lipofectamine™ 2000. On the contrary, 4T1 murine breast cancer cells express the murine TNF-α gene and therefore there is no need for transfection with such pCAm plasmid. 
19. A penicillin streptomycin solution (between 0.5 and 1.0 mL) is added to cell culture media (100 mL) to prevent bacterial contamination. A final concentration of penicillin (50–100 IU/mL) and streptomycin (50–100 μg/mL) must be obtained. 
20. A 1-h incubation involving murine expressing TNF-α plasmid and Lipofectamine™ 2000 has been also carried out obtaining similar transfection efficiencies. 
21. Lipid-siRNA duplexes were stable during the sonication process as observed by gel electrophoresis after phenol extraction of the serum proteins. 
22. The results from these experiments show that the amount of TNF-α produced by cells after 48 h with lipid-siRNA conjugates combined with 10% FCS differ from those transfection processes performed in the absence of the *pre-binding* strategy. In this sense, a 40–50% inhibition of the TNF-α production compared with the scrambled control was observed only if siRNAs were transfected using the *pre-binding* approach. Interestingly, the nature of lipid pendent groups at the 3′- and 5′-termini of the passenger strand is also important in the inhibition of the TNF-α expression. The strongest inhibition (80% in 4T1 cells) was observed when a siRNA duplex was modified with the hydrophobic lipid chain C14 at the 3′-termini of the sense strand (Figs. [5](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig5) and [6](clbr://internal.invalid/OEBPS/html/485053_1_En_8_Chapter.xhtml#Fig6)).