# RNA Interference: Story and Mechanisms RNA 干扰：故事和机制

Mouldy Sioud

## Abstract 摘要

The discovery that gene expression can be silenced by exogenously introduced double-stranded RNAs into cells unveiled a hidden level of gene regulation by a variety of small RNA pathways, which are involved in regulating endogenous gene expression, defending against virus infections, and protecting the genome from invading transposons, both at the posttranscriptional and epigenetic levels. All endogenous RNA interference pathways share a conserved effector complex, which contains at least an argonaute protein and a short single-stranded RNA. Such argonaute-RNA complexes can repress the transcription of genes, target mRNA for site-specific cleavage, or block mRNA translation into proteins. This review outlines the history of RNAi discovery, function, and mechanisms of action. For comparison, it also touches on CRISPR interference.

<span style=color:blue>基因表达可被外源性引入的双链RNA静默，揭示了一种隐藏的基因调控水平，通过各种小RNA途径参与调控内源基因表达、抵御病毒感染，并在转录后和表观遗传水平上保护基因组免受侵入的跳跃子。所有内源RNA干扰途径共享一个保守的效应器复合物，其中至少包含一个靶向蛋白和一个短的单链RNA。这样的靶向蛋白-RNA复合物可以抑制基因的转录，使mRNA靶向特定位点剪切，或阻断mRNA翻译成蛋白质。本综述概述了RNAi发现、功能和作用机制的历史。为了比较，它还涉及CRISPR干扰。</span>

## 1 Introduction

RNA silencing is a highly conserved gene regulation mechanism in which small RNA molecules silence gene expression by either suppressing transcription (transcriptional gene silencing) or by triggering a sequence-specific degradation of target mRNA, a process known as posttranscriptional gene silencing or RNA interference (RNAi). Although Fire’s Nature paper is considered a key breakthrough in the history of RNAi technology, it should be noted that in 1990 researchers described for the first time that RNA could potentially suppress gene expression in plants. Indeed, Napoli and Jorgensen were the first to report an RNAi type of phenomenon in plants 1990. By overexpressing the gene for chalcone synthase, an enzyme responsible for the production of anthocyanin pigments, the authors wanted to generate more purple transgenic petunia flowers. Surprisingly, some of the transgenic petunia lost both endogenous and transgene expression. This phenomenon was described as “cosuppression” and can be induced by single copy transgene that integrates into the genome in multicopy arrays. Later in 1992, a similar phenomenon was observed by Romano and Macino in *Neurospora crassa*.

<span style=color:blue>RNA沉默是一种高度保守的基因调控机制，小RNA分子通过抑制转录（转录基因沉默）或触发目标mRNA的序列特异性降解，即被称为转录后基因沉默或RNA干扰（RNAi）的过程，来沉默基因表达。尽管Fire在自然杂志上的论文被认为是RNAi技术历史上的重要突破，但值得注意的是，在1990年，研究人员首次描述了RNA可能在植物中抑制基因表达的可能性。事实上，Napoli和Jorgensen于1990年首次报告了植物中RNAi类型的现象。通过过度表达编码花青素色素产生的酶——花翠素合酶的基因，作者希望产生更多紫色的转基因矮牵牛花。令人惊讶的是，一些转基因矮牵牛花失去了内源和转基因的表达。这种现象被描述为“共抑制”，可以由单拷贝转基因引入多拷贝基因组中而诱发。后来在1992年，Romano和Macino在*神经孢子菌*中观察到了类似的现象。</span>

In 1995, Guo and Kemphues were the first to report RNA silencing in animals. The authors reported that sense RNA was as effective as antisense RNA for suppressing the expression of *par-1* gene in worm. *Par-1*, a gene required for establishing polarity in *Caenorhabditis* (C) *elegans* . At that time, antisense was one of the most attractive strategy for inhibiting gene expression. As a control, the authors used the sense par-1 RNA that would not hybridize to par-1 transcript. Interestingly, the degradation of par-1 mRNA was still observed. Although the results were interesting, the mechanism of mRNA degradation remained unknown.

In 1998, Fire and Mello showed in the worm *C. elegans* that double-stranded RNA (dsRNA), but not single-stranded RNA (ssRNA), was involved in gene silencing and coined the term “RNA interference, RNAi” [[4](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR4)]. The authors thought that the results of Guo and Kemphues showing that introduction of sense RNA leads to gene silencing could have been due to the contamination of the RNA preparation by dsRNA resulting from the activity of bacteriophage RNA polymerases. They purified the sense and antisense RNAs, then directly compared their effects on the *unc-22* gene, one of a set genes identified using classical genetics that affect muscle function in *C. elegans*. Overall, their data showed that exogenously introduced dsRNA reduces the expression of homologous mRNAs in *C. elegans*, and that RNAi may have a catalytic component. While these findings establish a new concept for the effects of RNA on gene silencing by highlighting a role for dsRNA, the mechanism of unwinding dsRNAs to promote the degradation of target mRNAs remained unresolved. It was later discovered that dsRNA molecules were processed into shorter intermediates dsRNAs and then into single-stranded RNAs that can ultimately base pair to the mRNA targets and induce their cleavage [[5](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR5)]. These RNA intermediates were called small interfering (si) RNAs and proved to be the main effectors of RNAi induction in mammalian cells. In conclusion, in RNAi, a dsRNA is involved and may originate from viral infection (exogenously), inverted-repeat transgenes, or from transcription of nuclear genes (endogenously).

## 2 The Interferon Response Hindered the Application of RNAi in Mammalian Cells

At the beginning, the use of RNAi was limited to flies, worms, and plants, as the introduction of long dsRNA into mammalian cells induces an interferon response that triggers a general inhibition of translation, abrogating the specificity of RNAi. However, in 2001 Tuschl and colleagues showed that duplexes of 21-nucleotide RNAs can bypass the interferon pathway and induce the degradation of their specific mRNA targets and thereby inhibiting the resulting protein expression [[6](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR6)]. These experiments were performed in cancer cell lines that may lack the receptors that recognize small RNAs. In this respect, several groups were able to show that certain siRNA sequences are potent activators of the immune system [[7](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR7)–[9](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR9)]. The activation of innate immunity by chemically made siRNAs is sequence-dependent and is normally mediated by Toll-like receptors (TLRs). We and others have demonstrated that the ribose sugar backbone are both necessary and sufficient for TLR7 and TLR8 recognition and that short ssRNA only requires several uridines in close proximity to render them effective immunostimulators [[10](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR10), [11](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR11)]. Fortunately, modifying siRNAs with as few as two 2′-OMe modifications at any position in the siRNA can eliminate the immunogenicity that can occur with unmodified siRNAs in immune cells [[12](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR12)]. Other 2′-ribose modifications such as 2′-F and 2′-deoxy can also reduce immune activation; however, the effects are not as pronounced as 2′-OMe RNAs [[12](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR12)]. Similarly, unmodified CRISPR guide RNAs induced type I interferon production in vitro in peripheral blood mononuclear cells , and that substitution of 2′OMe residues eliminated this response [[13](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR13)]. Interestingly, 2′-modified RNAs can function as TLR antagonists [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR14), [15](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR15)].

## 3 Molecular Mechanisms RNAi Induced by dsRNAs

Early in vitro work by Hannon, Zamore, Tuschl, and colleagues showed that the RNAi active protein complex contains 25 nt-long small RNAs that were homologous to the target mRNAs [[5](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR5), [16](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR16)]. Shortly thereafter, several laboratories were able to identify enzymes and cofactors in RNAi pathways using genetic and biochemical methods. These efforts revealed that dsRNAs are cleaved by the enzyme Dicer, a RNase III family member, to generate 21–23-nt shorter RNA duplexes (Fig. [1](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig1)). Dicer is a conserved protein among eukaryotes and can have multiple homologs with distinct functions. Like all RNase III enzymes, Dicer leaves 2-nt 3′-overhangs and 5′ phosphate groups [[17](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR17)]. Some organisms, including mammals and nematodes, have only a single Dicer enzyme that functions in the biogenesis of both miRNAs and siRNAs whereas other organisms divide the labor among multiple Dicer proteins. For example, two distinct enzymes are involved in parallel pathways for small regulatory RNA biogenesis in *Drosophila*. Dicer-2 processes long-double-stranded precursors and generates siRNAs, while Dicer-1 processes pre-miRNAs into mature miRNAs [[18](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR18), [19](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR19)].

![../images/485053_1_En_1_Chapter/485053_1_En_1_Fig1_HTML.png](clbr://internal.invalid/OEBPS/images/485053_1_En_1_Chapter/485053_1_En_1_Fig1_HTML.png)

**Fig. 1**

Schematic representation of gene silencing by double-stranded RNAs. The RNAi pathway is experimentally activated by introducing double-stranded RNA (dsRNA) to silence specific genes. Long dsRNA is cut into 19–21 bp siRNA fragments by Dicer and then into a multi-protein complex termed RNA-induced silencing complex (RISC) where the sense strand with high 5′-end stability is eliminated by the nuclease argonaute 2 (Ago2), resulting in strand separation. Synthetic siRNAs are directly loaded into the Ago2 RISC complex. Subsequent to strand separation, the sense strand guides the RISC to recognize and cleave complementary mRNA sequences. Gene silencing is a result of nucleolytic degradation of the targeted mRNA by Argonaute 2, an RNase H enzyme. Cleaved mRNA molecules are rapidly degraded by cellular nucleases. Following dissociation, the RISC is able to recycle and cleave additional mRNA molecules. Unlike chemically made siRNAs, hairpin RNAs (hsiRNAs) produced from plasmid vectors in cell nucleus are processed by Dicer in the cytoplasm before entering the RNAi pathway

In the next step, the siRNA duplexes are loaded onto an Argonaute protein , which along with Dicer and the trans-activation response RNA binding protein (TRBP) form the effector RNA-induced silencing complex (RISC). Subsequent to strand separation based on asymmetric unwinding [[20](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR20), [21](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR21)], the antisense strand guides the activated RISC to the complementary site in the target mRNA, resulting in mRNA cleavage (Fig. [1](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig1)). In mammals, the catalytic activity of RISC is mediated by Ago2 protein [[22](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR22)]. This slicer activity of the Ago-2 is very precise and cleaves in a position 10 and 11, counting from the 5′end, generating products with a 5′-monophosphate and a 3′-hydroxyl termini. Once the target mRNA is degraded by cellular nucleases, the RISC complex is recycled and free to cleave additional targets. Eukaryotic Argonaute protein consists of three conserved domains: PAZ, MID, and PWI. The PAZ domain binds the 3′end of the single-stranded RNA [[23](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR23)]. The Mid domain folds into a binding pocked that anchors the 5′ phosphate of the small RNA. Therefore, it is a strict requirement for the siRNAs to be 5′ phosphorylated to enter into the RISC pathway, and siRNAs that lack a 5′ phosphate are rapidly phosphorylated by an endogenous kinase prior to loading into the RISC complex. Analysis of the crystal structure of a siRNA guide strand associated with an Ago2 PIWI domain identified a seed sequence (nucleotides 2–8) that directs target mRNA recognition by RISC [[23](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR23), [24](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR24)]. It should be noted that the interactions between the small RNA and Ago2 protein are mediated by the sugar-phosphate backbone; as a result, the nucleotides of the guide RNA are free to base-pair with target mRNA. In humans, there are four Argonaute proteins and only Ago2 is catalytically active [[22](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR22)]. In flies and humans, the passenger strand cleaved by Ago2 is removed from the RISC-loading complex by the endonuclease C3PO, known as translin [[25](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR25)]. Although the mechanisms of gene silencing by dsRNAs are well studied, additional factors may contribute to strand selection by RISC and cleavage activity. Moreover, competition of coding and non-coding RNAs for Argonaute binding can influence gene silencing .

## 4 MicroRNAs: Masters of Gene Regulation

Although researchers demonstrated RNAi functionality using exogenous dsRNAs, RNAi is also induced by the endogenously present single-stranded hairpin microRNAs (miRNAs) (miRNAs) in cells [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR26)]. To date, miRNAs are the most abundant species of noncoding RNAs in mammalian cells, and they have fundamental roles in regulating gene expression at the posttranscriptional level [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR26), [27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)]. In some cases, RNAi can also confer resistance to virus or other pathogen infection. Notably, the first miRNAs were discovered through forward genetic screens in worms. Indeed, Ambros and colleagues described in 1993 a new mechanism of gene regulation in *C. elegans* through small non-protein-coding RNA lin-4, which was found to have complementary sequence to the 3′ untranslated region of lin-14 mRNA [[28](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR28)]. In both plants and animals, cloning was the initial strategy of large-scale miRNA discovery [[27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)]. Additionally, several groups developed bioinformatics approaches for miRNA identification [[29](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR29)]. According the miRBase database (release 22.1, [http://www.mirbase.org/](http://www.mirbase.org/)), there is currently 2656 human mature miRNA sequences that have the capacity to regulate more than two-thirds of protein-coding genes in humans and are involved in every cellular process known so far [[27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)].

In contrast to siRNAs, most miRNA genes are transcribed in the nucleus by RNA polymerase II, and the primary transcripts known as primary miRNAs (pri-miRNAs) are capped and polyadenylated [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR26), [27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)]. Before their export to the cytoplasm for further processing and silencing, nuclear pri-miRNAs must be recognized and cleaved (Fig. [2](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig2)). These respective tasks are carried out by the proteins Drosha and DGCR8 (known as Pasha in invertebrates), which together constitute the microprocessor complex [[26](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR26), [27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)]. As with Dicer, Drosha proteins belong to the RNase III family. The endonucleolytic activity results in the cleavage of the stem structure of pri-miRNAs and the release of the pre-miRNAs, which have a hairpin-like structure. Thereafter, Drosha-generated pre-miRNAs are exported from the nucleus to the cytoplasm by the nuclear export protein Exportin-5, which binds miRNA hairpin duplex along with GTP-bound form of the cofactor Ran, and releases it in the cytoplasm after hydrolysis of GTP [[27](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR27)]. In the cytoplasm, pre-miRNAs are cleaved further by Dicer to generate 22-nt miRNA duplexes containing mismatches and having a two nucleotides 3′ overhang. Subsequently, the miRNA duplex assembles with Ago2 protein to form the effector RISC complex after which the mature miRNA strand is selected (Fig. [2](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig2)). Depending on the homology to mRNAs, miRNA can lead either to mRNA cleavage (complete homology) or to repression of translation (partial homology). In flies and mammals few miRNAs are reported to lead to cleavage of their targets, whereas in plants most miRNAs exhibit complete homology with their corresponding target mRNA genes , resulting in target cleavage [[30](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR30), [31](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR31)]. Plant miRNA genes are generally not located within protein-coding genes but comprise their own RNA polymerase II-dependent transcriptional units. This contrasts with animal miRNAs, which sometimes appear to be processed from introns of protein-coding genes [[32](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR32)]. In plants, many predicted miRNA targets encode regulatory proteins, suggesting that plant miRNAs are master regulators [[30](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR30)]. Notably, miRNAs and siRNAs have much in common. Both are 20–24-nucleotides long and processed from longer RNA precursors by Dicer-like ribonucleases, and both are incorporated into ribonucleoprotein silencing complexes in which the small RNAs, through their base-pairing potential, guide target gene repression. However, the fundamental difference between the two small RNA classes is the nature of their precursors; siRNAs are processed from long, double-stranded RNAs, whereas miRNAs are processed from single RNA molecules that include imperfect stem-loop secondary structures as illustrated in Fig. [2](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig2).

![../images/485053_1_En_1_Chapter/485053_1_En_1_Fig2_HTML.png](clbr://internal.invalid/OEBPS/images/485053_1_En_1_Chapter/485053_1_En_1_Fig2_HTML.png)

**Fig. 2**

Schematic representation of miRNA silencing pathway. miRNAs are encoded in the genome and are transcribed to yield primary transcripts (pri-miRNAs), which are processed by Drosha with the aid of its partner DGCR8 protein (Pasha) to generate short precursor miRNAs (pre-miRNAs). These precursor miRNAs are then exported from the nucleus to the cytoplasm, where they are further processed by Dicer to yield miRNA duplexes that were unwound to single-stranded mature miRNA and then loaded into the miRNA-induced silencing complex (miRISC). Unwinding of the complex seems to occur so rapidly after duplex formation due to the Ago2 presence in the complex with Dicer and TRBP. The loaded strand RNA guides the RISC complex to recognize target mRNA sequences. A perfect homology allows Ago-cleavage of mRNA, whereas central mismatches promote repression of mRNA translation

## 5 piRNAs: Guardians of the Germline

The Piwi-interacting RNAs (piRNAs) are the most recently discovered class of small non-coding RNAs, and they bind to the PIWI subclade of Argonaute proteins [[33](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR33)]. One well-characterized Piwi/piRNA function is the silencing of retrotransposons, both at the posttranscriptional and epigenetic levels, as well as of other genetic elements in germlines, particularly those during spermatogenesis [[34](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR34)]. Fly piRNAs were initially termed repeat-associated small interfering RNAs (rasiRNAs) because they were found to map to transposons and repetitive elements [[35](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR35)]. Unlike rasiRNAs in flies, the majority of piRNAs in mammals arise from un-annotated regions of the genome devoid of transposons [[36](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR36)]. Currently there are about 23,500 piRNAs identified in the human genome, a number that is quite comparable to that of protein-coding mRNAs (around 20,000), suggesting that piRNAs may have several important roles in gene regulation that remain to be investigated.

Unlike miRNA and siRNA, piRNAs are typically longer, with a length of 26–31, originate from single-stranded RNA precursors, and do not require Dicer for maturation (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig3)). Most piRNAs are transcribed by RNA polymerase II as long transcripts which are then exported to the cytoplasm and processed into smaller sequences (mature piRNAs). In *D. melanogaster*, the nuclear export of piRNA precursors is mediated by UAP-56 that has been previously implicated in mRNA splicing and export [[37](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR37)]. Another factor that has been important in the context of piRNAs biogenesis is the mitochondrial surface protein Zucchini [[38](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR38)]. This protein has endonuclease activity for single-stranded RNA and likely processes the piRNA precursors, possibility at the 5′end. After 5′ processing, the 5′U/piRNA is incorporated into the MID domain of the Piwi protein, followed by 3′end processing, which likely involves an unknown trimmer activity as well as the action of methyl-transferase Pimet. Mature piRNAs are 5′ monophosphated and 2′-*O*-methyl modified in the 3′ terminal, a unique and specific feature of all piRNAs [[39](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR39)].

![../images/485053_1_En_1_Chapter/485053_1_En_1_Fig3_HTML.png](clbr://internal.invalid/OEBPS/images/485053_1_En_1_Chapter/485053_1_En_1_Fig3_HTML.png)

**Fig. 3**

Mechanisms of piRNA biogenesis in *D. melanogaster.* The piwi-interacting RNA pathway is involved in defense against transposable elements in the germline. Antisense precursor piRNAs are synthesized from repetitive elements by RNA polymerase II and then exported from the nucleus to nuage granules, where piRNA biogenesis is believed to occur [[36](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR36)]. These piRNA precursors are cleaved by the endonuclease Zucchini, located in the outer membrane of mitochondria, generating the 5′end of the future piRNAs. Then they are trimmed to short fragments, and their 3′ ends are 2′-methylated and then loaded onto aubergine (Aub), which cleaves complementary transposon transcripts to generate sense piRNA fragments. Argonaute 3 (Ago3) uses the sense piRNA fragments to bind and cleave additional complementary antisense transposon transcripts to generate more antisense piRNAs, thus creating a feedback loop known as the ping-pong cycle. This amplification step generates new piRNAs that are identical in sequence to the piRNA that initiated the cycle

The *D. melanogaster* genome encodes three Piwi proteins, Piwi, Aubergine (Aub), and Ago3, all of which are required for male and female fertility [[40](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR40)]. As shown in Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig3), Piwi and Aub show a strong preference for sequences with a 5′uridine (U) within the antisense to transposons, whereas Ago3 shows enrichment for adenine (A) at position 10 within the sense to transposons, suggesting that Ago3 piRNAs are generated by a new mechanism known as the “ping-pong cycle” of amplification [[40](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR40)–[42](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR42)]. In this cycle, Aub-bound piRNAs recognize a complementary transposon and induce endonuclease cleavage of the target. Such slicing generates the 5′ end of new sense piRNAs. This cytoplasmic loop, which is also found in zebrafish, links piRNA amplification to posttranscriptional target silencing [[41](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR41)]. Additionally, mature piRNAs in the cytoplasm form a complex with Piwi proteins and migrate back into the nucleus, reaching their target transcripts and mobilizing the silencing machinery to block the transcription of transposable elements via DNA methylation (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig3)). Although the role of piRNAs in silencing transposon repeat elements is well established, recent reports have shown that piRNAs can also regulate gene expression and perfect sequence homology between the piRNA and its target mRNA is not required for efficient repression by deadenylation [[36](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR36)]. Apart from the gonads, piRNAs and PIWI proteins may be expressed in various somatic cell types, albeit typically at lower levels than those observed in the germline [[43](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR43)]. Similar to miRNAs , a higher number of piRNAs are expressed in tumors compared to normal tissues [[44](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR44)].

## 6 CRISPR Interference

As indicated above, RNAi plays an essential role in many plant biological processes, including developmental timing and patterning, transposon control, DNA methylation, and chromatin modification. Although its role in human immunity is not known yet, RNAi constitutes the primary natural plant immune system against viruses [[45](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR45)]. The plant RNAi defense system is highly specific because it will only target nucleic acids that are identical in sequence to the triggering dsRNAs. In addition to viral infections, RNAi protects the genome from genetics parasites such as transposons and retrotransposons. These mobile nucleic acid elements can cause diseases and alter the genome in a number of different ways.

By analogy to RNAi-mediated plant protection against pathogen attack, the *C*lustered *R*egularly *I*nterspaced *S*hort *P*alindromic *R*epeats (CRISPR) loci and CRISPR-associated (Cas) proteins provide acquired immunity against viruses and other mobile genetic elements in bacteria and archaea. These CRISPR sequences are part of the bacteria’s immune system and provide endogenous adaptive immunity in approximately 40% of bacterial genomes and 70% of sequenced archaeal species [[46](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR46), [47](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR47)]. Immunological memories of phages and plasmids are stored in the CRISPR array as short spacer sequences that intercalate between repeats and specify the targets of CRISPR-Cas immunity. The locus consists of CRISPR-associated (Cas) genes in operons in addition to the spacer-repeat array. These repeats were initially discovered in the 1980s in *E. coli* [[48](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR48)], but their function was not uncovered until 2007 by Barrangou and colleagues, who demonstrated that the *Streptococcus thermophilus* strains that survived initial phage challenge were resistant to subsequent phage challenge and had incorporated sequences from the phage genome into their CRISPR arrays [[47](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR47)]. When a new infection occurs, the spacer-repeat array is expressed as a long precursor pre-CRISPR RNA (pre-crRNA), and the transcript is processed into individual spacer repeat units as crRNAs, while the *cas* genes are transcribed and translated into proteins. Additionally, the *Cas* genes are transcribed into proteins. Such crRNAs associate with and direct Cas nucleases (e.g., Cas9) to cleave foreign nucleic acids complementary to the spacer sequence. Cleavage of the viral or plasmid target DNA prevents infection (Fig. [4](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#Fig4)). Target recognition requires base pairing between the crRNA and the target and a PAM sequence adjacent to the crRNA-binding region. The requirement of a PAM site prevents cleavage of the CRISPR loci, which lacks a PAM site [[47](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR47)]. The PAM sequences allow the discrimination between self and non-self, a crucial characteristic of all immune systems.

![../images/485053_1_En_1_Chapter/485053_1_En_1_Fig4_HTML.png](clbr://internal.invalid/OEBPS/images/485053_1_En_1_Chapter/485053_1_En_1_Fig4_HTML.png)

**Fig. 4**

Schematic representation of CRISPR-Cas systems. When a phage infects a bacterium, it injects its DNA into the bacterium which responds by producing Cas enzymes to cut the phage DNA into small fragments and then incorporate them into its own genome at the CRISPR locus. During the second infection, the bacterium that survived the first infection activates the CRISPR locus to generate small RNA molecules and Cas protein enzymes. After RNA processing, each small RNA molecule associates with the Cas effector protein and guides the enzyme to cleave the invading phage DNA, thus blocking the infection. Cas genes code for important endonuclease enzymes in this pathway One of these, Cas9, is a nuclease that cuts nucleic acid (DNA or RNA)

## 7 A Versatile Genome Editing Technology

CRISPR/Cas9 genome editing is a relatively new nucleic acid-based technique that relies on high-performing nucleic acids to produce highly efficient and specific alterations to the genome. The technology has been adapted for genome editing applications in eukaryotic cells and is an emerging tool because it allows for some more flexibility compared to other gene-editing tools. Moreover, the technique is incredibly precise, cheap, and easy. The CRISPR-Cas system described so far falls into two major classes, which are further divided into a total of six types. Class 1 uses a multi-subunit effector complex and consists of CRISPR type I, II, and IV. Class 2 employs a single effector nuclease and consists of CRISPR type II, V, and VI [[49](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR49)]. CRISPR type II is known by Cas9 nuclease, a multidomain protein that is able to bind crRNA and cleave corresponding target DNA and thus represents the most convenient format to use in eukaryotic cells. Indeed, in a landmark 2012 paper, Doudna, Chapentier, and Jinek showed that they could use CRISPR-Cas 9 system to introduce specific mutations in any genome [[50](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR50)]. When loaded with both crRNA and tracrRNA (or an artificial chimera of the two), the wild-type Cas9 can site-specifically introduce double-stranded DNA breaks (DSBs). The resulting DSB will either generate nonspecific mutations knocking out a gene through the error-prone NHEJ (non-homologous end joining) pathway. Alternatively, if a donor template with homology to the target locus is supplied, the DSB may be repaired by the homology-directed repair (HDR) pathway allowing for precise replacement mutations to be made [[51](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR51), [52](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR52)]. By directing the cells toward one of these repair mechanisms, CRISPR/Cas9 can be used to generate knockouts or knock-ins in cells. In addition to gene editing, the technology was also adapted for mRNA targeting (knockdown) with high specificity [[53](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR53)]. This system relies on an enzyme called Cas13a, which complexes with guide RNA, which binds to complementary messenger RNA. The better the match between the guide RNA and the messenger RNA, the more effective the Cas13a’s gene silencing action.

In contrast to previous tools for genome editing such as zinc-finger nucleases and transcription activator-like effector nucleases, CRISPR does not require engineering of new enzymes for each new target. With respect to specificity, high-fidelity Cas9 variants including eSpCas9, SpCas9-HF1, HypaCas9, and xCas9 were developed [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR54)]. These Cas9 protein variants combined with the use of chemically modified gRNAs have increased the specificity of genome editing [[54](clbr://internal.invalid/OEBPS/html/485053_1_En_1_Chapter.xhtml#CR54)]. Although RNAi was adopted as gene silencing technique first, CRISPR has surpassed RNAi in popularity due to several advantages including specificity and versatility. Drs Emmanuelle Chapentier and Jennifer Doudna who developed the CRISPR technology are awarded the Nobel Prize in Chemistry 2020.

## 8 Conclusions

Analysis of RNAi in a variety of organisms revealed that RNAi is a widespread natural phenomenon that is conserved across fungi, plants, and animals. The process is induced not only by exogenous siRNAs but also by several endogenous small RNA species such as microRNAs and piwi-interacting RNAs. While miRNAs participate actively in all cellular processes, piwi-interacting RNAs (piRNAs) have a major function in the repression of transposable elements in the germline. Recently, CRISPR/Cas9 technology has evolved as the most powerful approach to generate genetic models both for fundamental and preclinical research. Not only can scientists use such technology to silence genes by knocking them out, they can also introduce desired genes into the genome. The choice remains with the user to perform knockout, knock-ins, or knockdown experiments, making CRISPR interference extremely versatile.

## Acknowledgments

This work was supported by grants from the Norwegian Cancer Society and the Southern and Eastern Norway Regional Health Authority (Helse Sør-Øst).