# Selection of Chemical Modifications in the siRNA Seed Region That Repress Off-Target Effect

Yoshiaki Kobayash, Kengo Miyamoto, Misako Aid and Kumiko Ui-Tei

## Abstract

RNA interference mediated by small interfering RNA (siRNA) has been widely used as a procedure to knock down the expression of an intended target gene with perfect sequence complementarity. However, siRNA often exhibits off-target effects on genes with partial sequence complementarities. Such off-target effect is an undesirable adverse effect for knocking down a target gene specifically. Here we describe the powerful strategy to avoid off-target effects without affecting the RNAi activity by the introduction of DNA or 2′-*O*-methyl modifications in the siRNA seed region. These two types of chemical modifications repress off-target effects through different molecular mechanisms.

## 1 Introduction

RNA interference (RNAi) is induced by small interfering RNA (siRNA), a duplex composed of the guide and passenger strands of 21-nucleotide (nt)-long RNAs with 2-nt 3′ overhangs. The siRNA is loaded onto Argonaute (AGO) protein , which is a core component of the RNA-induced silencing complex (RISC) [[1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR1)–[3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR3)]. Subsequently, one of the two strands (passenger strand) is ejected and the other strand (guide strand) remains loading on AGO. Then, the guide strand base pairs with its target mRNA with perfect sequence complementarity, leading to repression of gene expression through the cleavage by AGO (Fig. [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig1)). Synthetic siRNAs have become a powerful tool not only for studying gene function but also for applying therapeutics. Because the first siRNA drug was approved by FDA and EMA in 2018 [[4](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR4)], the developmental research for therapeutic applications expected to become popular.

![485053_1_En_2_Fig1_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_2_Fig1_HTML.png)

**Fig. 1**

Schematic presentation of the mechanisms of siRNA-mediated RNAi and sequence-dependent off-target effects. SiRNA, a duplex composed of the guide and passenger strands, is loaded onto AGO protein. Passenger strand is ejected and guide strand remains loading on AGO. The mRNA with exact complementary sequence with siRNA guide strand is repressed by RNAi through its cleavage by AGO protein. On the other hand, the sequence-dependent off-target effect is induced for the unintended mRNAs with sequence complementarities with siRNA seed region (2–8 nucleotides from the 5′ end of the guide strand) in their 3′UTRs through translational repression (*see* **Note** [**4**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14))

RNAi efficiencies of siRNAs are widely different depending on their sequences, and only a limited fraction of siRNAs, which satisfy the functional siRNA sequence rules, is effective in mammalian cells [[5](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR5)]. Meanwhile, the siRNA often exhibits off-target effects when the siRNA seed region (2–8 nts from the 5′ end of the guide strand) base pairs with the unintended genes [[6](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR6)–[10](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR10)] (Fig. [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig1)). The off-target effect is considered to be induced by the similar molecular mechanism with miRNA-mediated RNA silencing, which differs from the mechanism of RNAi. Furthermore, the degree of off-target effect is correlated positively with the thermodynamic stability in base-pairing between the seed region of the siRNA guide strand and the unintended mRNA [[11](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR11)]. Therefore, siRNA with low seed-target stability may be a promising tool for target-specific RNAi with little off-target effect. However, the number of the functional siRNA sequences with lower seed-target stability is substantially small.

Thus, we have established the methods to overcome such sequence-dependent limitations using two types of chemical modifications (Fig. [2](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig2)): One type of chemical modification is DNA and the other is 2′-*O*-methyl (2′-OMe). The results of reporter assay (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3)), structural simulation (Fig. [4](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig4)), and microarray (Fig. [5](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig5)) reveal that the seed-dependent off-target effect is able to be reduced by the introduction of DNA and 2′-OMe modifications in the siRNA seed region without reducing RNAi activity through different molecular mechanisms: DNA modification in the siRNA seed region decreased the thermodynamic stability in the seed-target base-pairing stability [[12](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR12)], and 2′-OMe modification induced steric hindrance in the duplex formation on the AGO protein [[13](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR13)]. These chemical modifications may be useful tools for potential therapeutic applications.

![485053_1_En_2_Fig2_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_2_Fig2_HTML.png)

**Fig. 2**

Flow for selecting functional siRNAs with reduced sequence-dependent off-target effect by two types of chemical modifications. (**a**) From all of human mRNA sequences, 56,375,087 of 23-mer sequences are selectable. This number of siRNAs is set as 100%. (**b**) Functional siRNA sequences can be selected by using the four rules described in **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14). (**c**) The siRNAs with sequences containing low seed–target *T*m values (*see* **Note** [**2**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)) for both strands are selected for reduction of off-target effects. (**d**) Using DNA or 2′-OMe modification in the siRNA seed region makes it possible to overcome the sequence limitations of siRNA for reducing seed-dependent off-target effects through different molecular mechanisms: reduction of base-pairing stability by DNA modification, and steric hindrance by 2′-OMe modification

![485053_1_En_2_Fig3_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_2_Fig3_HTML.png)

**Fig. 3**

Measurement of RNAi and off-target activities by dual luciferase reporter assays . (Top) Structures of the constructs for luciferase reporter assays. psiCHECK-gCM_VIM270 contains a CM sequence of the siRNA guide strand in the downstream of *Renilla luciferase* CDS (CDS). psiCHECK-gSM_VIM270, psiCHECK-pCM_VIM270, or psiCHEK-pSM_VIM270 contains three tandem repeats of the SM sequences of the guide strand, the CM and SM sequences of the passenger strand, respectively. The reporter assays were performed as follows: HeLa cells were co-transfected with each concentration of unmodified, DNA- or 2′-OMe-modified siVIM-270, with each psiCHECK construct and pGL3 control vector. Cells were harvested 24 h after transfection and relative luciferase activities were measured. (Bottom) Unmodified, DNA- or 2′-OMe-modified siVIM-270 sequences (left). Results of RNAi activities on the guide CM target using psiCHECK-gCM_VIM270, off-target activities on the guide SM target using psiCHECK-gSM_VIM270, RNAi activities on the passenger CM target using psiCHECK-pCM_VIM270 and off-target activities on the passenger SM target using psiCHECK-pSM_VIM270 by the unmodified or modified siVIM-270s. We used siGY441 as siRNA control (siCont). The data were averaged from three independent experiments, and the bar indicates the standard deviation

![485053_1_En_2_Fig4_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_2_Fig4_HTML.png)

**Fig. 4**

Computational prediction of the steric hindrance of 2′-OMe-modified siRNAs loaded on the AGO protein. (**a**) Crystal structure of unmodified 5′-AAA-3′ RNA with K566 and R792 of AGO protein shown by Schirle et al. [[28](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR28)], and the optimized structures of unmodified (**b**) and 2′-OMe-modified (**c**) RNAs with K566 and R792. (**d**) Superposition at C4′-C3′-O3′ of A3 of unmodified RNA with 2′-OMe-modified RNA

![485053_1_En_2_Fig5_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_2_Fig5_HTML.png)

**Fig. 5**

Microarray analysis of RNAi activities and off-target effects of unmodified, and DNA- and 2′-OMe-modified siRNAs. (**a**) Expression levels of the target vimentin gene in the cells transfected with the unmodified and modified siRNAs relative to that in the mock-transfected cells, respectively. (**b**) MA plots of microarray data of the cells transfected with unmodified, and DNA- and 2′-OMe-modified siRNAs, respectively. The vertical axis indicates the log2 fold change of signal intensity of each gene in each type of siRNA-transfected cells relative to that of mock (*M* value), and the horizontal axis indicates the averaged log10 signal intensity of each gene in mock and siRNA transfection (*A* value). The dark blue dots indicate the transcripts with seed-matched (SM) sequences in the 3′UTRs of the mRNAs, and the light blue dots indicate the other transcripts. (**c**) Cumulative distribution of microarray data of the cells transfected with unmodified, and DNA- and 2′-OMe-modified siRNAs, respectively. The horizontal axis indicates the M value of (**b**), and the vertical axis indicates the cumulative fraction of transcripts. The red line indicates the cumulative curve of SM transcripts, and the black line indicates the cumulative curve of the other non-SM transcripts. (**d**) Mean log2 fold-change of the downregulated SM transcripts compared to those of non-SM transcripts. The horizontal axis indicates the mean fold-change of off-target transcripts in the cells transfected with unmodified, and DNA- and 2′-OMe-modified siRNAs, respectively

## 2 Materials

### 2.1 Design and Preparation of siRNAs

1. siDirect2.0 software: [http://siDirect2.RNAi.jp/](http://sidirect2.rnai.jp/)

2. Guide strand sequence of siRNA against human vimentin gene (siVIM-270): 5′-UUGAACUCGGUGUUGAUGGCG-3′.

3. Passenger strand sequence against human vimentin gene (siVIM-270): 5′-CCAUCAACACCGAGUUCAAGA-3′.

   SiRNA against green fluorescent protein (siGY441), which does not have perfectly complementary sequences in complete-match (CM) and seed-match (SM) reporter constructs (*see* Subheading [2.2](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec4)), is used as a negative control.

4. Guide strand of siGY441: 5′-AUGAUAUAGACGUUGUGGCUG-3′.

5. Passenger strand of siGY441: 5′-GCCACAACGUCUAUAUCAUGG-3′.

6. Annealing buffer (2×): 40 mM NaCl, 20 mM Tris–HCl, pH 7.5.

7. 19% Polyacrylamide gel .

8. TBE buffer.

### 2.2 Luciferase Reporter Vector and Oligonucleotides with CM and SM Sequences

1. psiCHECK-1 vector (Promega).

2. Oligonucleotides with CM sequences of siVIM-270 guide strand (gCM) with cohesive XhoI/EcoRI ends: 5′-tcgaGCGCCATCAACACCGAGTTCAAGAG-3′ and 5′-aattCTCTTGAACTCGGTGTTGATGGCGC-3′.

3. Oligonucleotides with three tandem repeats of SM sequences of siVIM-270 guide strand (gSM) with cohesive XhoI/EcoRI ends: 5′-tcgaGAATGATGCACCAGGAGTTCAAGAAATGATGCACCAGGAGTTCAAGAAATGATGCACCAGGAGTTCAAGAG-3′ and 5′-aattCTCTTGAACTCCTGGTGCATCATTTCTTGAACTCCTGGTGCATCATTTCTTGAACTCCTGGTGCATCATTC-3′.

4. Oligonucleotides with three tandem repeats of CM sequences of siVIM-270 passenger strand (pCM) with cohesive XhoI/EcoRI ends: 5′-tcgaTCTTGAACTCGGTGTTGATGGCGAATCTTGAACTCGGTGTTGATGGCGAATCTTGAACTCGGTGTTGATGGCGAA-3′ and 5′-aattTTCGCCATCAACACCGAGTTCAAGATTCGCCATCAACACCGAGTTCAAGATTCGCCATCAACACCGAGTTCAAGA-3′.

5. Oligonucleotides with three tandem repeats of CM sequences of siVIM-270 passenger strand (pSM) with cohesive XhoI/EcoRI ends: 5′-tcgaAATGATGCACCAGGAGTTGATGGAATGATGCACCAGGAGTTGATGGAATGATGCACCAGGAGTTGATGG-3′ and 5′-aattCCATCAACTCCTGGTGCATCATTCCATCAACTCCTGGTGCATCATTCCATCAACTCCTGGTGCATCATT-3′.

### 2.3 Cell Culture

1. Human HeLa cells or other culture cells .

2. 24-Well plate (Sumitomo Bakelite).

### 2.4 Transfection and Luciferase Reporter Assay Reagents

1. pGL3-Control vector (Promega).

2. Lipofectamine 2000 reagent (Thermofisher).

3. Dual-Luciferase Reporter Assay System (Promega).

4. 1× passive lysis buffer (Promega).

### 2.5 Calculation of *T*m

The melting temperature (*T*m) values are calculated using the following formula using Δ*H* and Δ*S* thermodynamic parameters shown in Xia et al. [[14](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR14)] and Sugimoto et al. [[15](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR15)].

$$ {T}_{\mathrm{m}}=\left\{\left(1000\times \Delta H\right)/\left[A+\Delta S+\ln \left(\mathrm{Ct}/4\right)\right]\right\}-273.15+16.6\mathrm{log}\left[{\mathrm{Na}}^{+}\right]. $$

- Δ*H* (kcal/mol), sum of nearest neighbor enthalpy change.
- *A*, helix initiation constant (−10.8).
- Δ*S*, sum of nearest neighbor entropy change.
- *R*, gas constant (1.987 cal/deg/mol).
- Ct, total molecular concentration of strand (100μM).
- Fix [Na+] at 100 mM.

### 2.6 Microarray Analysis

1. RNeasy kit (Qiagen).

2. NanoDrop 2000 spectrophotometer (Thermo Scientific).

3. Bioanalyzer (Agilent).

4. Agilent one-color spike mix kit (Agilent).

5. Agilent SurePrint G3 human GE microarray (8 × 60 K, ver. 2.0, 3.0) (Agilent).

## 3 Methods

### 3.1 Designing and Preparing siRNAs with DNA or 2′-OMe Modifications in the Seed Region

Highly functional siRNAs for mammalian cells (*see* **Note** [**1**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)) can be designed using the web-based online software, siDirect 2.0. In this section, the procedure for selecting highly functional siRNA for human vimentin is shown as an example.

1. Open “[http://siDirect2.RNAi.jp/](http://sidirect2.rnai.jp/)” with any appropriate web browser.

2. Input the cDNA sequence of vimentin in a nucleotide sequence box. Or enter the accession number of human vimentin (NM_003380) into the upper blank box.

3. Click “retrieve sequence” to get the nucleotide sequence from GenBank ([https://www.ncbi.nlm.nih.gov/genbank/](https://www.ncbi.nlm.nih.gov/genbank/)).

4. Click the “Options: [click here]” button to set optional parameters if necessary (*see* **Note** [**2**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

5. Click “design siRNA ,” and you can get result.

6. Synthesize siRNAs shown in Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3) with or without DNA and 2′-OMe modifications in the siRNA seed region chemically, respectively (*see* **Note** [**3**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

7. Mix the solutions containing same amounts of guide and passenger strand RNAs, and add the same volume of 2× annealing buffer, and anneal by incubating the mixture at 95 °C for 3 min, at 37 °C for 30 min, and at 25 °C for 60 min.

8. Annealed products may be checked using 19% polyacrylamide gel electrophoresis in TBE buffer, which can separate 21-nt double-stranded siRNA with 2-nt 3′ overhangs from single-stranded RNA .

### 3.2 Preparing Luciferase Reporters with CM and SM Sequences

1. Both strands of DNA oligonucleotides for gCM and gSM reporters, or pCM and pSM reporters are mixed, respectively, and add the same volume of 2× annealing buffer. They are annealed by incubating at 95 °C for 3 min, at 37 °C for 30 min, and at 25 °C for 60 min.

2. Insert the annealed oligonucleotides into the XhoI/EcoRI restriction enzyme sites within 3′ UTR region of *Renilla* luciferase gene in psiCHECK-1, and name each plasmid vector, such as psiCHECKgCM_VIM270, psiCHECK-gSM_VIM270, psiCHECK-pCM_VIM270, and psiCHECK-pSM_VIM270, respectively (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3)). RNA silencing activities of the guide strand and passenger strand are measured using psiCHECKgCM_VIM270 and psiCHECK-pCM_VIM270, respectively. The seed-dependent off-target effects of the guide strand and passenger strand are measured using psiCHECK-gSM_VIM270 and psiCHECK-pSM_VIM270, respectively (*see* **Note** [**4**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

### 3.3 RNA Silencing Activity Assay Using the Dual Luciferase Reporter System

RNA silencing activity is measured using dual luciferase reporter assay system.

1. Inoculate a human HeLa cell suspension (1.0 × 105 cells/mL) into a well of 24-well culture plate 1 day before transfection.

2. Transfect 0.0005, 0.005, 0.05, 0.5, 5 nM of each siRNA (*see* **Note** [**5**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)), 0.1μg of pGL3-Control vector encoding the firefly luciferase gene, and 0.1μg of each psiCHECK-gCM_VIM270, psiCHECK-gSM_VIM270, psiCHECK-pCM_VIM270, and psiCHECK-pSM_VIM270 reporter construct simultaneously into HeLa cells using 2μL of Lipofectamine 2000.

3. Lyse the transfected cells with 1× passive lysis buffer 24 h after transfection (*see* **Note** [**6**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

4. Measure luciferase activity using the Dual-Luciferase Reporter Assay System.

5. Calculate the percentage of the *Renilla* luciferase activity normalized by firefly luciferase activity (*Renilla* luciferase activity/firefly luciferase activity × 100) for unmodified siRNA, and siRNA with DNA (*see* **Note** [**7**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)) or 2′-OMe (*see* **Note** [**8**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)) modification in the seed region.

### 3.4 Microarray Analysis

1. Inoculate HeLa cells in each 2 wells of 24-well plate at 1 × 105 cells/mL.

2. Transfect 50 nM of each siRNA into HeLa cells using 2μL of Lipofectamine 2000. Prepare mock-transfected cells treated with the transfection reagent in the absence of siRNA as a control.

3. At 24 h post-transfection, purify total RNA from the transfected cells using an RNeasy kit.

4. Measure the concentration of purified RNA using a NanoDrop 2000 spectrophotometer and confirm its quality by a Bioanalyzer (*see* **Note** [**9**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

5. Synthesize cDNA from 1μg of each total RNA sample using an Agilent one-color spike mix kit.

6. Hybridize the cDNA product to an Agilent SurePrint G3 human GE microarray according to the manufacturer’s protocol.

7. Calculate the reduction of the expression levels of target vimentin mRNAs in the cells transfected with each siRNA relative to that in the mock-transfected cells, respectively.

8. Generate *M* (log ratio) and *A* (mean average) (*MA*) plot and cumulative distribution using microarray data of each siRNA by package in R-studio software to evaluate the RNA silencing activity (*see* **Note** [**10**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)) and off-target effect (*see* **Note** [**11**](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Sec14)).

## 4 Notes

1. SiRNAs satisfied the following four sequence conditions simultaneously are considered to be functional: (a) A/U at the 5′-terminus of the siRNA guide strand, (b) G/C at the 5′-terminus of the siRNA passenger strand, (c) AU richness at the 5′ one-third of the guide strand, (d) absence of a long GC stretch (>9 nt). Bioinformatics analysis revealed that 14.7% of all the 23-mer subsequences in human mRNA satisfy functional siRNA sequence rules [[16](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR16)] (Fig. [2a, b](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig2)).

2. Using options, other siRNA selection algorithms established by Reynolds et al. [[17](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR17)] or Amarzguioui and Prydz [[18](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR18)] can be selected. Furthermore, the highly effective siRNA with reduced seed-dependent off-target effect is selectable using *T*m value in the seed-target duplex below 21.5 °C. *T*m value is a good benchmark to express the thermodynamic stability of the duplex between siRNA seed region and target mRNA. Calculated *T*m value of siRNA seed-target duplex in protein-free condition discriminates siRNA with almost weak off-target effect from that with strong off-target effect [[11](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR11)]. Specificity of siRNA sequence in human, mouse, or rat can be also checked. Other options which impose the number of contiguous GC or AT nucleotides or GC content are also available. 

3. DNA is a modified form of RNA with hydrogen at 2′-position of the pentose sugar. The 5′ one-third of siRNA is capable of replacement with DNA without substantial loss of RNAi activity [[12](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR12)]. 2′-OMe is a well-known C2′-modified RNA that is known to increase stability in the serum and thermodynamic stability in base-pairing, and also considered to abrogate immunogenicity [[19](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR19)–[23](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR23)]. 2′-OMe modification is tolerated at multiple positions in the siRNA guide strand because of its small size, comparable to the 2′-OH of natural RNA [[24](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR24)]. 

4. In RNAi, a target mRNA is reduced by cleavage of target mRNA by AGO2. However, miRNA-mediated RNA silencing is induced by translational repression [[25](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR25)–[27](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR27)]. 

5. It is preferable to perform reporter assays at a series of multiple concentrations of each siRNA to calculate the IC50 of its RNA silencing activity quantitatively. 

6. If the cell debris is formed from transfected cells, it is preferable to remove the debris clearly before addition of 1× passive lysis buffer to measure the luciferase activity correctly. 

7. Unmodified siVIM-270 repressed the *Renilla* luciferase activity derived from psiCHECK-gCM for siVIM-270 (psiCHECK-gCM_VIM270) in a dose-dependent manner (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3)). The calculated IC50 was 2 pM (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Tab1)). DNA-modified siVIM-270 repressed the *Renilla* luciferase activity at the almost equivalent level to that of unmodified siVIM-270, particularly at high concentrations (0.5 or 5 nM), and their IC50 was 4 pM (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Tab1)). The off-target effects on the SM target of the guide strand were also measured using psiCHECK-gSM_VIM270 (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3)). Unmodified siVIM-270 showed strong off-target effects (IC50 = 10 pM), but DNA-modified siVIM-270 reduced the off-target effects (IC50 = 295 pM) (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Tab1)). This may be because thermodynamic stability in base-pairing of DNA-RNA duplex is weaker than that of RNA duplex [[13](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR13)]. In fact, the calculated *T*m value of the RNA duplex between the seed region of unmodified siVIM-270 and target RNA was 31.4 °C, and that of DNA-RNA heteroduplex between the seed region of DNA-modified siVIM-270 and target RNA was 3.1 °C. Furthermore, it is possible that the passenger strand also induces unintended RNAi and seed sequence-dependent off-target effects. Then dual luciferase reporter assays were also performed using psiCHECK-pCM_VIM270 and psiCHECK-pSM_VIM270, respectively. However, almost no or little inhibitory effects on both CM and SM targets of the passenger strand were observed (Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3)). Because we used siRNA satisfying the four functional siRNA sequence conditions, such siRNA is expected to be easily unwound from the 5′ terminus of the guide strand, but not that of the passenger strand. 

8. The 2′-OMe-modified siVIM-270 repressed the *Renilla* luciferase activity derived from psiCHECK-gCM for siVIM-270 (psiCHECK-gCM_VIM270) at the almost equivalent level to that of unmodified siVIM-270, and its IC50 was 10 pM (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Tab1)). The off-target effects on the SM target of the guide strand of 2′-OMe-modified siVIM-270 measured using psiCHECK-gSM_VIM270 was significantly reduced, even though this modification is well-known modification to enhance base-pairing stability [[19](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR19)]. It is known that the phosphates of RNA oligonucleotides interact with the amino acid side chains of human AGO protein [[28](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR28), [29](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR29)]. Therefore, 2′-OMe-modified single-stranded RNA structure on the AGO protein was examined computationally (Fig. [4](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig4)). The 2′-OMe modification in the siRNA seed region at position 3 was essentially stable on the AGO protein. However, RNA structure of the nucleotide adjacent to the 2′-OMe-modified nucleotide was changed from that of unmodified RNA (Fig. [4a–d](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig4)), suggesting that 2′-OMe modification in the siRNA seed region at least at position 3 from 5′ end may disturb base-pairing between siRNA guide strand and its target or off-target mRNAs. But, as shown in Fig. [3](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig3), the expression of the CM target was significantly repressed, suggesting that the non-seed region may compensate the incomplete seed-target base pairing for inducing RNAi on the CM target. 

9. Particularly, it is recommended that the value of RNA Integrity Number (RIN), a quantitative measurement of RNA quality, of total RNA sample for microarray application is high. High-quality samples with RIN values higher than 9 are best for microarray application, but the reliable results are also obtained from the sample with the value as low as 7.0. 

10. Unmodified siVIM-270 repressed the expression level of vimentin mRNA to 11%. DNA- and 2′-OMe-modified siVIM-270 repressed vimentin mRNA expression at almost equivalent level to 13% and 19%, respectively (Fig. [5a](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig5)). 

11. The seed-dependent off-target effect of each siRNA using microarray data can be evaluated by analyzing the expression profiles of genes with seed matched sequences in their 3′UTRs, since siRNA off-target effect is mainly induced by the mechanism similar to miRNA-mediated RNA silencing [[25](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR25)–[27](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#CR27)]. Off-target effects on mRNAs having complementary sequences with siRNA seed region in their 3′UTRs were examined. The obvious off-target effects were observed by the introduction of unmodified siVIM-270. However, DNA-modified siVIM-270 showed reduced off-target effects. On the other hand, 2′-OMe-modified siRNA showed little off-target effects (Fig. [5b–d](clbr://internal.invalid/OEBPS/html/485053_1_En_2_Chapter.xhtml#Fig5)). 


**Table 1**

IC50s of unmodified, DNA-modified, or 2′-OMe-modified siVIM-270 for RNAi activity or off-target effect measured by reporter assay

| siRNA  | RNAi (pM) | Off-target effect (pM) |
| :----- | :-------- | :--------------------- |
| RNA    | 2         | 10                     |
| DNA    | 4         | 295                    |
| 2′-OMe | 10        | ∞                      |

∞ indicates that the IC50 is higher than 5000 pM