# siRNA Design and GalNAc-Empowered Hepatic Targeted Delivery siRNA 设计和 GalNAc 增强的肝靶向递送

Mei Lu, Mengjie Zhang, Bo Hu and Yuanyu Huang 

## Abstract 摘要

Small interfering RNA (siRNA) is a clinically approved therapeutic modality, which has attracted widespread attention not only from basic research but also from pharmaceutical industry. As siRNA can theoretically modulate any disease-related gene’s expression, plenty of siRNA therapeutic pipelines have been established by tens of biotechnology companies. The drug performance of siRNA heavily depends on the sequence, the chemical modification, and the delivery of siRNA. Here, we describe the rational design protocol of siRNA, and provide some modification patterns that can enhance siRNA’s stability and reduce its off-target effect. Also, the delivery method based on *N*-acetylgalactosamine (GalNAc)-siRNA conjugate that is widely employed to develop therapeutic regimens for liver-related diseases is also recapitulated.

## 1 Introduction

RNA interference (RNAi) based on small interfering RNA (siRNA) is not only widely used to regulate gene expression in basic research areas but also represents a clinically approved therapeutic platform that can be investigated for treating diverse life-threaten diseases, such as metabolic disorders, genetic or orphan diseases, infectious diseases, and cancers. Mature siRNA can be generated by both chemical synthesis and processing from short hairpin RNA (shRNA) or long double-stranded RNA with Dicer (Fig. 1). Once the siRNA duplex is introduced into the cytoplasm of cells, it will be incorporated into the RNA-induced silencing complex (RISC). Then the sense (passenger) strand of siRNA is released from RISC, and the antisense (guide) strand of siRNA will recognize the target complementary mRNA in a complete match manner. Finally, the target mRNA will be cleaved by Argonaut protein in RISC. The cleaved mRNA fragments are further degraded by cellular 5′ and 3′ exonucleases (Fig. .1). In contrast to the recognition mode of “complete match with target mRNA,” microRNA (miRNA), another representative member of RNAi family, modulates gene’s expression in a “partial match” or “seed match” manner. In which way, only the seed region of the miRNA (position of 2–8 from 5′ end) needs to match with the targeted gene. Nucleic acid therapeutics based on both siRNA and miRNA are investigated for treating diseases currently, although siRNA is more potent and popular for drug development. siRNA and miRNA are usually considered to be generated exogenously and endogenously, respectively. Hence, the sequences of siRNA are artificially designed, while the sequences of miRNA are selected from naturally existing miRNA library. Actually, the activity of siRNA molecules is highly dependent on the sequence since not all double-stranded RNA molecules that completely matched with target mRNA can trigger efficient gene silence. Many factors contribute to the overall activity of siRNA, such as G and C content, the stability of the 3′ terminus of the sense strand (it will determine which strand of the duplex will be loaded into RISC), the nucleotide (base) type of the 5′ and 3′ end of antisense strand. Taken together, rational design of siRNA is essentially required.

![485053_1_En_6_Fig1_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig1_HTML.png)

**Fig. 1** Working mechanism of siRNA. Synthetic siRNA or Dicer-processed siRNA is incorporated into RNA-induced silencing complex (RISC). The sense (passenger) strand is removed and the antisense (guide) strand will recognize the full complementary mRNA. Then Argonaut protein in RISC cleaves the target mRNA, and the protein translation is blocked

In addition, the issues of metabolic stability, specificity, and off-target effect are the key determinants of drug performances in vivo. Both the sequence and chemical modification may contribute to these issues. For instance, siRNA with UA/UA and/or CA/UG paring in the sequence is more sensitive to RNase attacking, and modifications in these bases can enhance its resistance to enzyme’s degradation. All the mismatch type (e.g., A-G, A-C, A-A, and G-U, G-A, G-G, etc.), mismatch position (from 5′-end to 3′-end of siRNA, and on coding region or untranslated region of mRNA), and mismatch number (one or two or more mismatches) could affect the activity and specificity of siRNA. Chemical modification may also change the strand selection of RISC, thus contributing to enhancing the specificity of siRNA. The off-target effect of siRNA may occur through several different mechanisms, including: (1) passenger strand of siRNA binds to undesired target mRNA; (2) the seed regions of passenger or guide strand of siRNA recognize certain mRNA sequence, especially at the 3′-untranslated region (UTR) region, and silence gene’s expression in a “miRNA-like” way; (3) passenger or guide strand of siRNA bind to toll-like receptors (TLRs, e.g., TLR3), stimulate immune response, and affect downstream gene’s expression. Various chemical modification strategies have been investigated to reduce the off-target effect of siRNA. For example, 2′-*O*-methyl (OMe) modification at position 14 of the siRNA passenger strand can be used to abolish the activity of the passenger strand. Unlocked nucleic acid (UNA) and glycol nucleic acid (GNA) at position 7 of the siRNA guide strand dramatically reduced the “miRNA-like” off-target effect. Actually, recognition of toll-like receptors and stimulation of immune response directly caused the termination of Phase III study of Bevasiranib, the first-ever clinically investigated siRNA therapeutic, in 2009. Disturbance of the expression of undesired target genes resulted from the seed regions of both strands of siRNA may also trigger significant hepatotoxicity in animals . Therefore, rational chemical modifications are extremely important for drug development.

Moreover, insufficient delivery of siRNA to the targeted tissue and cell significantly hampered siRNA therapeutic development for a long period. Because the cell membrane is primarily made up of zwitterionic and negatively charged phospholipids, and shows a negative potential (−40 to −80 mV) across the cell membrane, it is difficult for the negatively charged siRNA to cross cell membrane by itself. Hence, various delivery platforms, including chemically synthesized materials (such as liposomes, polymers, dendrimers, and inorganic nanoparticles), biological agents (such as peptides, antibodies, aptamers , CpG, 3-Way Junction, and exosomes), and physical approaches (such as electroporation, microneedle poking, hydrodynamic injection, and microfluidic extrusion), have been developed to deliver siRNA to desired tissues and cells (Fig. 2). Among them, *N*-Acetylgalactosamine (GalNAc)-siRNA conjugate exhibits excellent hepatocyte-targeted delivery efficiency and safety profile in vivo (Fig. 2d, e, g) . GalNAc can be recognized by asialoglycoprotein receptor (ASGPR) that is conserved across species and highly expressed by hepatocytes, and can mediate clathrin-involved endocytosis. It takes only 15 min for GalNAc from binding to ASGPR to being internalized into the cell and re-displaying on cell surface. After thoroughly exploring the effects of valence of oligosaccharides, distance among the targeting moieties, and size of the particle on the binding performance, researchers reached a consensus that triantennary GalNAc with a mutual distance of ~20 Å exhibits the highest affinity with ASGPR. By conjugating three GalNAc moieties to the siRNA with proper linker chemistry, siRNA can be efficiently delivered to hepatocytes via subcutaneous injection. Benefiting from the enhanced stabilization modification, GalNAc-siRNA conjugate can remain in the circulation system and cytoplasm for more than 100 days, thus permitting a long-term gene silencing and therapeutic effect in human beings. The reduction profile could be well maintained for more than 6 months or even 1 year, after a single dose of GalNAc-siRNA, as demonstrated by abundant clinical data.



**Fig. 2**

![485053_1_En_6_Fig2_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig2_HTML.png)Schemes and structures of representative siRNA delivery platforms. Schemes of (**a**) lipid nanoparticles, (**b**) DPC2.0™ or EX-1™, (**c**) TRiM™, (**d**) GalNAc-siRNA conjugates, (**e**) GalXC™ and (**f**) LODER™, respectively. (**g**) Chemical structures of the key component(s) of the discussed delivery platforms, including Dlin-DMA, Dlin-MC3-DMA, C12-200, cKK-E12, MLP-based DPC2.0, PLGA-based LODER, and GalNAc-siRNA conjugates. *DPC*, Dynamic PolyConjugates; *TRiM*, Targeted RNAi Molecule; *LODER*, LOcal Drug EluteR; *PLGA*, poly(lactic-co-glycolic) acid; *MLP*, membrane-lytic peptide, with the amino sequence NH-LIGAILKVLATGLPTLISWIKNKRKQ-COOH; *CDM*, carboxylated dimethyl maleic acid; *PEG*, polyethylene glycol; *NAG*, *N*-acetylgalactosamine (GalNAc)

In this chapter, we will introduce the siRNA design method with online-accessible tools, discuss chemical modification strategies, and recapitulate the experimental design process of GalNAc-mediated liver-targeted siRNA delivery. It may provide a brief roadmap for researches from both academic institute and pharmacy industry.

## 2 Materials

### 2.1 Software, Database, and Equipment

1. DNAMAN 8, and/or online alignment tool (http://molbiol-tools.ca/Alignments.htm). 
2. “LALIGN,” use following website: https://embnet.vital-it.ch/software/LALIGN_form.html (*see* **Note** **1**). 
3. Online siRNA design tools (*see* **Note** **2**). 
4. NCBI database, including the sub-databases of “Gene” (https://www.ncbi.nlm.nih.gov/gene/) and “Nucleotide” (https://www.ncbi.nlm.nih.gov/nucleotide/). 
5. NCBI tool of Blastn: https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome 
6. Extended Nucleic Acid Sequence Massager: https://www.cmbn.no/tonjum/seqMassager-saf.htm 
7. A personal computer with Microsoft office and access to the World Wide Web. 
8. Microvolume Spectrophotometers. 
9. Real-time PCR machine. 
10. Multi-Detection Microplate Reader. 
11. Luminex 100 system.

### 2.2 Reagents

1. siRNA, including GalNAc-siRNA conjugate. 
2. Lipofectamine 2000 (Thermo Fisher Scientific, USA). 
3. DMEM medium: Dulbecco’s Modified Eagle’s Medium (DMEM) without supplement. 
4. HEK293A medium: DMEM medium supplemented with 1% penicillin-streptomycin, and 10% fetal bovine serum. 
5. Perfusion solution 1: Hank’s Balanced Salt Solution (HBSS) supplemented with 0.5 mM EDTA, pH = 8. 
6. Perfusion solution 2: DMEM medium supplemented with 0.8 mg/mL collagenase type I. 
7. Primary mouse hepatocyte medium: DMEM medium supplemented with 10% fetal bovine serum, but without adding antibiotics. 
8. Opti-MEM medium (Thermo Fisher Scientific, USA) with no supplement. 
9. Trypsin. 
10. Trizol. 
11. Cell culture plates. 
12. Serological pipettes. 
13. Reverse transcription kit. 
14. Real-time PCR kit (UltraSYBR Mixture). 
15. RNAlater. 
16. Dual luciferase reporter system: the primary plasmid vectors of siQuant and psiCheck. 
17. Luciferase detection kit. 
18. Plasmid isolation kit. 
19. Anesthetic: 4% Chloral hydrate solution (w/v). 

### 2.3 Cells and Animals

1. HEK293A originally derived from human embryonic kidney cells . Culture HEK293A cells in HEK293A medium. 
2. Male C57BL/6 mice (age 6–8 weeks). 

## 3 Methods

RNAi therapeutic companies may establish their own siRNA design algorithm and develop their confidential software. Here we just introduce how to design siRNAs with online accessible tools. Firstly, design siRNAs with online tools that are accessible for public (Table [1](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#Tab1)). Then evaluate the activity, stability, and off-target effect in vitro. After narrowing down the number of siRNA candidates, apply chemical modifications to the siRNAs, and evaluate the performance of siRNA again. Then conjugate three or four (typically three) GalNAc moieties to the siRNA with proper linker chemistry. Finally, inject the GalNAc-siRNA conjugate into the mice subcutaneously and assess the in vivo gene silence and toxicity.

**Table 1** Selected online siRNA design tools and their features

<div class="c-table-scroll-wrapper__content c-table-scroll-wrapper__fade c-table-scroll-wrapper__fade--transparent" data-component-scroll-wrapper=""><table class="data last-table"><thead class="c-article-table-head"><tr><th class="u-text-left "><p>Tool</p></th><th class="u-text-left "><p>Developer (owner)</p></th><th class="u-text-left "><p>Website</p></th><th class="u-text-left "><p>Rules or features</p></th></tr></thead><tbody><tr><td class="u-text-left "><p>siDESIGN Center</p></td><td class="u-text-left "><p>Dharmacon/Horizon Discovery</p></td><td class="u-text-left "><p>https://horizondiscovery.com/products/tools/siDESIGN-Center</p></td><td class="u-text-left "><p>(a)&emsp;30–52% G/C content</p><p>(b)&emsp;At least 3 “A/U” bases at positions 15–19 (sense strand)</p><p>(c)&emsp;Absence of internal repeats (Tm of potential internal hairpin is &lt;20&nbsp;°C)</p><p>(d)&emsp;An “A” base at position 19 (sense strand)</p><p>(e)&emsp;An “A” base at position 3 (sense strand)</p><p>(f)&emsp;A “U” base at position 10 (sense strand)</p><p>(g)&emsp;A base other than “G” or “C” at 19 (sense strand)</p><p>(h)&emsp;A base other than “G” at position 13 (sense strand)</p></td></tr><tr><td class="u-text-left "><p>siDirect 2.0</p></td><td class="u-text-left "><p>University of Tokyo</p></td><td class="u-text-left "><p>http://sidirect2.rnai.jp/</p></td><td class="u-text-left "><p>(a)&emsp;A or U at position 19 (guide strand, from 5′ to 3′)</p><p>(b)&emsp;Four to seven A/Us in nucleotide positions 1–7 (A/U&nbsp;≥&nbsp;57%)</p><p>(c)&emsp;G/C at position 19</p><p>(d)&emsp;No long GC stretch (≤9 GC)</p><p>(e)&emsp;Low thermodynamic stability in the duplex formed between the siRNA 7-nt seed region and the target mRNA (Tm&nbsp;≤&nbsp;21.5&nbsp;°C)</p></td></tr><tr><td class="u-text-left "><p>Block-iT RNAi Designer</p></td><td class="u-text-left "><p>Invitrogen/Thermo Fisher Scientific</p></td><td class="u-text-left "><p>https://rnaidesigner.invitrogen.com/rnaiexpress/setOption.do?designOption=sirna</p></td><td class="u-text-left "><p>Unspecified algorithm; partially based on Tuschl’s rules and patterns</p></td></tr><tr><td class="u-text-left "><p>DSIR</p></td><td class="u-text-left "><p>Ecole des Mines de Paris</p></td><td class="u-text-left "><p>http://www.bioinfo.ensmp.fr/dsir/</p></td><td class="u-text-left "><p>Based on linear models over two sets of features: (1) the nucleotides present at each position; (2) the global content of the siRNA in short motifs. Short asymmetric motifs contain as much relevant information for potency prediction as the nucleotide preferences for particular positions</p></td></tr><tr><td class="u-text-left "><p>GenScript siRNA Target Finder</p></td><td class="u-text-left "><p>GenScript</p></td><td class="u-text-left "><p>https://www.genscript.com/tools/sirna-target-finder</p></td><td class="u-text-left "><p>Following factors are taken into account:</p><p>(a)&emsp;Internal stability of siRNA duplex</p><p>(b)&emsp;Specificity of the siRNA target sequence</p><p>(c)&emsp;Internal secondary structure of the sense and anti-sense strands</p><p>(d)&emsp;Sequence complexity</p><p>(e)&emsp;Single nucleotide polymorphism</p><p>(f)&emsp;Other adjustable algorithm parameters include GC content, asmmetrey of double strands of the siRNA duplex, sequence region, etc.; other adjustable off target filter include study organism, study organ, size of seed region and functional alignment</p></td></tr><tr><td class="u-text-left "><p>RNAi Design Tool</p></td><td class="u-text-left "><p>IDT</p></td><td class="u-text-left "><p>https://sg.idtdna.com/site/order/designtool/index/DSIRNA_CUSTOM</p></td><td class="u-text-left "><p>Support vector machine (SVM)-based selection</p></td></tr><tr><td class="u-text-left "><p>siDRM</p></td><td class="u-text-left "><p>University of Minnesota</p></td><td class="u-text-left "><p>http://c1.accurascience.com/siDRM/index.php</p></td><td class="u-text-left "><p>High sensitivity rule sets and fast rule sets based on disjunctive rule merging (DRM) analysis. Innate immune responses, cell toxic effects and off-target activities are taken into consideration</p></td></tr><tr><td class="u-text-left "><p>siRNA Selection Server</p></td><td class="u-text-left "><p>Whitehead Institute</p></td><td class="u-text-left "><p>http://jura.wi.mit.edu/bioc/siRNAext/</p></td><td class="u-text-left "><p>Functionality prediction limited to Tuschl or user-defined pattern-based selection; incorporates design rules that published before 2004</p></td></tr><tr><td class="u-text-left "><p>OptiRNA</p></td><td class="u-text-left "><p>University of Nebraska-Lincoln</p></td><td class="u-text-left "><p>http://optirna.unl.edu/</p></td><td class="u-text-left "><p>Support vector machine (SVM)-based selection; mRNA secondary structure is taken into account; does NOT run BLAST searches</p></td></tr></tbody></table></div>

*Ref.(s)* reference(s), *NA* not available

### 3.1 Primary Mouse Hepatocyte Isolation

Isolate primary mouse hepatocytes from C57BL/6 mice according to the protocol reported previously and described as following.

1. Anesthetize the animals by intraperitoneally injecting a saline solution containing 80 mg/kg Ketamine and 5 mg/kg Xylazine. 

2. Prepare the animal for surgery (clean/sanitize). 

3. Immobilize the animal with tape. Dissect the abdominal cavity, move the gastric system to the right, and expose liver. 

4. To start perfusion, insert the catheter into the vena cava through the right atrium. The flow rate should be 5 mL/min. 

5. Create back pressure by blocking the visceral vena cava with a finger when the blood is drained out. Perfuse for 5–7 min.

6. Position a lamp a few inches above the animal to maintain the cavity temperature of 37 °C throughout the procedure. 

7. Switch the tubing from perfusion solution 1 to perfusion solution 2, and continue perfusion for 7–8 min (flow rate 5 mL/min).

8. Collect the liver into perfusion solution 2, and cut the liver sac to release the hepatocytes.


9. Add DMEM medium and pass the liver suspension through a cell strainer. 

10. Spin cells at 800 rpm for 5 min. Wash the pellet twice with DMEM medium. 

11. Count cells with a hemocytometer and culture in primary mouse hepatocyte medium. 


### 3.2 siRNA Design and Screening

#### 3.2.1 mRNA (cDNA) Selection

1. Identify the target gene that we intend to silence. 

2. Determine the accession number(s) of the mRNA sequence(s) of the target gene, and/or download the mRNA (cDNA) sequence(s) in a FASTA format. 

3. Analyze mRNA (cDNA) sequence(s) and determine the sequence(s) that will be used to design siRNA according to the workflow shown in Fig. 3(*see* **Note** **3**). 


![485053_1_En_6_Fig3_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig3_HTML.png)

**Fig. 3** Workflow of mRNA (cDNA) sequence selection

#### 3.2.2 Generation of siRNA Sequence List

1. Submit mRNA (cDNA) sequence of interest, or the accession number of the target gene, to the online siRNA design tools. The available online tools and related information are summarized in Tables 1 and 2. 

2. Establish a series of siRNA design algorithms by analyzing hundreds and thousands of siRNAs. The silencing activity, selectivity, off-target effect, as well as the mRNA secondary structure, alternative splicing, or the motif sequences may potentially trigger immune response when they were taken into accounts. One or more design tools can be used when submitting mRNA (cDNA) sequence. 

3. siRNA list exported by the selected tool is automatically generated. Typically, 3–5 siRNA candidates that rank on the top of the list can be selected for further evaluation. High-ranking siRNAs means their scores are relatively high because the features of these sequences match better with the algorithm than other sequences (*see* **Notes** **4** and **5**). 

4. Synthesize the siRNAs without chemical modification (*see* **Note** **6). 


**Table 2**

Representative siRNA designing algorithms and their attributes

| Authors            | Tool            | Position (5′–3′, cDNA form) | 1    | 3    | 6    | 10   | 11   | 13    | 16   | 19   |
| :----------------- | :-------------- | :-------------------------- | :--- | :--- | :--- | :--- | :--- | :---- | :--- | :--- |
| Reynolds et al.    | siDESIGN Center | Preferred                   |      | A    |      | T    |      | A/C/T |      | A/T  |
| Ui-Tei et al.      | siDirect 2.0    | Preferred                   | G/C  |      |      |      |      |       |      | A/T  |
|                    |                 | Unpreferred                 | A/T  |      |      |      |      |       |      | G/C  |
| Amarzguioui et al. | NA              | Preferred                   | G/C  |      | A    |      |      | T     | C    | A/T  |
|                    |                 | Unpreferred                 | T    |      |      | T    |      |       |      | G    |
| Jagla et al.       | NA              | Preferred                   | G/C  |      |      | A/T  |      |       |      | A/T  |
| Hsieh et al.       | NA              | Preferred                   |      |      |      |      | C/G  | A     | G    | T    |
| Unpreferred        |                 |                             | C    |      | A/T  |      |      | G     |      |      |

*Ref*. reference, *NA* not available

#### 3.2.3 Activity Evaluation

1. Evaluate the activities of siRNA candidates via either luciferase reporter system or reverser transcription quantitative PCR (RT-qPCR), or both (*see* **Note** **7**). 

2. For luciferase reporter assay, insert the target sequences into the siQuant reporter vector according to standard molecular clone procedures [[9](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#CR9)] (*see* **Note** **8**). 

3. Transfect siRNAs into the Hek293A cells with Lipofectamine 2000 at various transfection concentrations (at least three concentrations, e.g., 10 nM, 1 nM, and 0.1 nM; or more than seven concentration gradients that enable building a concentration-activity curve and calculating the IC50), together with the reporter vectors (*see* **Note** **9**). 

4. 24 h later, harvest the cells and prepare cell lysates. Then, transfer 10μL cell lysates to the reader plate, and add the substrates of Firefly and Renilla luciferases into the samples sequentially. Detect the luminance signal with Multi-Detection Microplate Reader. 

5. For RT-qPCR assay, transfect the siRNAs into the cells that endogenously express the target gene at desired concentrations (typically no less than three concentrations, e.g., 100, 20, and 5 nM). 

6. 24 or 48 h later, harvest the cells and extract the total RNA according to the standard procedures. Verify the quality of total RNA by separating in agarose gel and analyzing with Microvolume Spectrophotometers. 

7. Prepare the cDNA with 1μg total RNA in a 20μL standard reaction solution, then use 80μL ddH2O to dilute the cDNA solution. 

8. Perform the quantitative PCR with a 20μL reaction solution containing 5μL of cDNA template by using a PCR machine . 

9. Finally, calculate the inhibition efficiency with the Ct values by normalizing to the control group (e.g., mock without transfection siRNA, or scrambled siRNA-treated sample, or untreated sample) (*see* **Notes** **10-12**). 


#### 3.2.4 Off-Target Effect Evaluation

1. Assess the off-target effect of siRNAs with either psiCheck reporter system or RNA-seq technology. 

2. For psiCheck-based off-target effect evaluation, transfect siRNAs into Hek293A at concentration gradients ranged from 50 nM, 0.5 nM, 0.25 nM, 0.125 nM, 0.0625 nM, 0.0313 nM, 0.0156 nM, 0.0078 nM, 0.0039 nM, 0.002 nM, 0.001 nM to 0.0005 nM. 

3. Record the enzymes’ activities of Renilla and Firefly luciferases with Multi-Detection Microplate Reader. Then generate the concentration-activity curves and calculate the IC50s with GraphPad Prism software. 

4. With this reporter-based assay, the off-target effects resulted from complete-match of the passenger strand with the target or from seed-match of either the guide or the passenger strand with the target can be quickly determined. 

5. Alternatively, apply high throughput RNA-seq, a newly developed technology to evaluate the expression inhibition of numerous endogenously expressing genes both in vitro and in vivo (Fig. 7b). 

6. In this circumstance, prepare total RNA with high quality from either cultured cells (cell lines or primary cells) or tissue samples. Perform the RNA-seq usually by Contract Research Organization (CRO). Analyze the data by ourselves (if feasible) or the researchers of CRO. Finally, figure out the expression levels of all on-target genes and those off-target transcripts. 

7. Assess the off-target effect resulted from the immune response, such as activation of the TLRs pathway, by measuring the cytokine levels and the expression of relevant genes (e.g., TNF-α, IL-6, IL-12, IFN-γ, IFIT genes). 


### 3.3 siRNA Chemical Modification

1. It is well known that chemical modification of siRNA is extremely essential for siRNA therapeutic development because it can significantly enhance siRNA’s capability of resistance to enzymes’ attacking, improve siRNA’s specificity, and reduce potential off-target effects. Sophisticated modification strategies have been proposed and investigated for both siRNA with canonical structure or Dicer-substrate siRNA (DsiRNA) (Fig. [4](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#Fig4)). Chemical structures of the representative modification building blocks are shown in Fig. 5. 

2. Select siRNA modification design from Fig. 4. Alternatively, test additional designs based on these patterns or other novel designs. 

3. Among these designs, 2′-fluoro (2′-F), 2′-*O*-methoxy (2′-OMe), DNA, phosphorothioate, and inverted base (idT/A) are primarily employed to enhance siRNA’s stability , while unlocked nucleic acid (UNA) and glycol nucleic acid (GNA) positioned at 7 of the antisense strand are used to reduce the off-target effect mediated by seed region (Fig. 7a). 


![485053_1_En_6_Fig4_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig4_HTML.png)

**Fig. 4** Examples of siRNA chemical modification design. GalNAc, *N*-acetylgalactosamine, can be placed at 3′-end or 5′-end of the sense strand. For DsiRNA, they can be positioned at the unpaired G-A-A-A nucleotides. Abbreviations: *2*′*-F* 2′-fluoro, *2′-OMe* 2′-methoxy, *UNA* unlocked nucleic acid, *GNA* glycol nucleic acid, *SS* sense strand, *AS* antisense strand

![485053_1_En_6_Fig5_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig5_HTML.png)

**Fig. 5** Chemical structures of siRNA modification building blocks. R=H or OH, for RNA or DNA, respectively

### 3.4 In Vivo Delivery of GalNAc-siRNA Conjugate

1. As aforementioned, GalNAc-siRNA conjugate constitutes one of the most powerful and successful delivery platforms for liver-targeted transportation of siRNA. Therefore, dozens of siRNA therapeutic pipelines have been established, aiming to treat diverse diseases including hyperlipidemia, hepatitis B, amyloidosis, etc. 

2. Display GalNAc moieties at the 3′-end or 5′-end of the sense strand of siRNA. Alnylam Pharmaceuticals, Silence Therapeutics, Arbutus Biopharma, etc., conjugate trivalent GalNAc at the 3′-end of siRNA sense strand, while Arrowhead Pharmaceuticals demonstrate there is no significant difference between 3′-conjugation and 5′-conjugation. Dicerna Pharmaceuticals position tetravalent GalNAc at the unpaired nucleotides of the DsiRNA structure. A challenging task for this field is to develop proprietary linker structures. A series of different linkers have been designed, and some of them can deliver oligonucleotides to hepatocytes efficiently. The linker chemistry developed by Alnylam is shown in Fig. 2g. 


### 3.5 In Vivo Activity of GalNAc-siRNA Conjugate

1. Subcutaneously (SC) administer GalNAc-siRNA conjugate into the mice (male C57BL/6, 6–8 weeks, weighing 18–22 g) at 0.5–10 mg/kg (*see* **Note** **13**). 

2. Sacrifice animals by cervical dislocation, harvest the liver tissues 3 days or 1 week after administration and keep them in RNA*later* (*see* **Note** **14**). 

3. Homogenize the tissues and extract the total RNAs according to the same protocol with that of the cultured cells. 

4. Prepare cDNAs and quantify the expression levels of the target gene by real-time qPCR (*see* **Note** **12**). 

5. Administer multiple doses of GalNAc-siRNA to achieve potent and durable gene silence and treatment effect. Examples of gene silence performances with single or multiple doses of GalNAc-siRNA are shown in Fig. 6. 

6. As an optional choice, quantify siRNAs exposed in liver tissue and load into the RISC by stem-loop reverse transcription followed by Taqman PCR (Fig. 7c, d). 


![485053_1_En_6_Fig6_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig6_HTML.png)

**Fig. 6** Gene silencing by GalNAc-siRNA conjugate targeting mouse transthyretin (mTTR) in C57BL/6 mice. (**a**) mRNA expression of mTTR in liver tissues after treated with single dose of GalNAc-siRNA conjugate (dark blue; 0.2, 1, and 5 mg/kg, *n* = 5). (**b**) Long-term reduction of TTR protein in mice during weekly subcutaneous dosing at 1 mg/kg. The TTR protein was analyzed with serum samples by ELISA (enzyme-linked immunosorbent assay). Data were shown as mean ± standard error. (The figure is adapted and modified from the literature. Copyright © 2014 American Chemical Society)

![485053_1_En_6_Fig7_HTML](https://raw.githubusercontent.com/zcgkiller/Pictures/main/Wechat/485053_1_En_6_Fig7_HTML.png)

**Fig. 7** The in vivo performances of GNA-modified GalNAc-siRNA conjugate targeting hydroxyacid oxidase 1 (HAO1). (**a**) The modification design of siRNA. (**b**) Off-target effect evaluated by RNA-seq in rat hepatocytes. (**c**, **d**) siRNA amount in liver tissue (**c**) and RISC (**d**) that quantified by stem-loop RT-qPCR. (**e**) Glutamate dehydrogenase (GLDH) levels measured with rat serum specimen. (**f**) H&E staining of the liver tissues collected at necropsy. Error bars represent standard deviation of the mean. **p* < 0.05. (The figure is adopted from the literature, Copyright 2018 Nature Publishing Group)

### 3.6 In Vivo Toxicity of GalNAc-siRNA Conjugate

1. According to the guidelines of toxicity assessment , a series of studies need to be performed for Investigational New Drug (IND) application. These assessments include but not limited to acute toxicity test, long-term toxicity test, genetic toxicity test, reproductive toxicity test, and carcinogenic toxicity test. Herein, we mainly discuss some preliminary toxicity assessments. 

2. Subcutaneously administer GalNAc-siRNA conjugate to toxicology-relevant animal species, such as rat and nonhuman primate (NHP), at toxicological doses (*see* **Notes** [**15**](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#Sec17) and [**16**](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#Sec17)). 

3. Collect plasma or serum samples at 3–6 h or 24 h after administration. 

4. Examine the cytokines, such as TNF-α, IFN-γ, IL-1β, IL-2, IL-5, IL-6, IL-12(p70), MCP-1, KC, GM-CSF using Luminex® assays (high-throughput multiplex bead-based assays) with Luminex 100 system or ELISA. 

5. Count the blood cell with whole blood samples to analyze the profiles of the key components in the blood, such as red blood cells, white blood cells, hemoglobin, hematocrit, and platelets. 

6. In another study, do not collect any sample within 2 weeks after siRNA injection. Instead, perform clinical observations for 2 weeks. In this case, record clinical symptoms, including animal appearance, eating, drinking, excretion, behavior, and response to stimulation during this process. Finally, as an optional choice, collect the blood samples, measure parameters revealing the liver and kidney functions (e.g., aspartate transaminase (AST), alanine transaminase (ALT), alkaline phosphatase (ALP), total bilirubin (TBIL), glutamate dehydrogenase (GLDH), serum creatinine (CREA), blood urea nitrogen (BUN)), and analyze cytokines and the blood cells. Evaluate the pathological changes of the tissue samples. 

7. An example of toxicity evaluation of GalNAc-siRNA conjugate is shown in Fig. [7e, f](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#Fig7). It has been demonstrated that GNA positioned at 7′ of siRNA guide strand can significantly erase the seed region-mediated off-target effect and therefore dramatically alleviate the hepatotoxicity. 


## 4 Notes

1. Use other alignment tools. 

2. Some commonly used online siRNA design tools are shown as follows:

   (a) siDESIGN Center ([https://horizondiscovery.com/products/tools/siDESIGN-Center](https://horizondiscovery.com/products/tools/siDESIGN-Center)).

   (b) siDirect 2.0 ([http://sidirect2.rnai.jp/](http://sidirect2.rnai.jp/)).

   (c) Block-iT RNAi Designer ([https://rnaidesigner.invitrogen.com/rnaiexpress/setOption.do?designOption=sirna](https://rnaidesigner.invitrogen.com/rnaiexpress/setOption.do?designOption=sirna)).

   (d) DSIR ([http://www.bioinfo.ensmp.fr/dsir/](http://www.bioinfo.ensmp.fr/dsir/)).

   (e) GenScript siRNA Target Finder ([https://www.genscript.com/tools/sirna-target-finder](https://www.genscript.com/tools/sirna-target-finder)).

   (f) RNAi Design Tool ([https://sg.idtdna.com/site/order/designtool/index/DSIRNA_CUSTOM](https://sg.idtdna.com/site/order/designtool/index/DSIRNA_CUSTOM)).

   (g) siDRM ([http://c1.accurascience.com/siDRM/index.php](http://c1.accurascience.com/siDRM/index.php)).

   (h) siRNA Selection Server ([http://jura.wi.mit.edu/bioc/siRNAext/](http://jura.wi.mit.edu/bioc/siRNAext/)).

   (i) OptiRNA ([http://optirna.unl.edu/](http://optirna.unl.edu/)). 

3. Alternatively, design siRNAs separately with the transcripts for targeting conserved regions across species. Combine siRNA lists reported by different tools, and analyze their sequence features to determine if there is any siRNA targeting the conserved regions.

4. Combine the siRNA sequences and verify if there is any siRNA that is reported by more than one tool when using multiple tools to design siRNA. Prefer siRNA(s) reported by more than one tool, especially if the algorithms are different from each other.

5. Typically, select less than ten siRNAs for further activity evaluation for basic research because higher than 80% silencing efficiency usually can be achieved within these siRNAs. There is no necessity to consider the intellectual property issues if we do not intend to submit a patent application. Otherwise, select dozens or even thousands of siRNAs for screening and evaluation.

6. Alternatively, chemically modified siRNA. Perform the following tests with the modified siRNAs. The advantage of using modified siRNA is saving time and cost. However, a potential risk is that we may miss some siRNAs with high activity when they are not modified. Because chemical modification may influence the activity although the modification chemistry is rationally designed and optimized.

7. If the library of siRNAs is not large, for instance, less than 20 siRNAs, directly use RT-qPCR for activity evaluation instead of using luciferase reporter system.

8. Evaluate the activity of siRNA with psiCheck reporter vector, because it is demonstrated that there is no significant difference between the siRNAs targeting the CDS region and those targeting the 3′-UTR, regarding to their silencing activities, as evaluated by reporter system [[9](clbr://internal.invalid/OEBPS/html/485053_1_En_6_Chapter.xhtml#CR9)]. 

9. Use other nucleic acid transfection reagents, e.g., Lipofectamine 3000 and Lipofectamine RNAiMAX. 

10. Typically, use a scrambled siRNA in both luciferase assay and PCR assays, and normalize the activity of tested siRNA to that of the scrambled siRNA. However, if the Ct values of untreated sample, mock sample, and scrambled siRNA-treated sample are consistent, use all the above negative controls for normalization. 

11. The sequence of a scrambled siRNA that is widely used: sense strand: UUCUCCGAACGUGUCACGUdTdT, antisense strand: ACGUGACACGUUCGGAGAAdTdT. Bioinformatics analysis shows that this siRNA target no transcript for human, mouse, rat, and monkey.

12. Determine the protein expression level with Western Blotting. In this case, extract total protein and perform the assay according to the well-established standard protocols.

13. Change the dosage according to requirements of the assay. If the activity of the tested siRNA is extremely high, use low dose. If the duration of single dose of siRNA is intended to be observed, use relatively higher dose. 

14. If the duration of siRNA potency is intended to be evaluated, choose longer period, e.g., 1 month, even half year. 

15. Determine the toxicological doses that are normally higher than the treatment doses by using dose range finding (DRF) study. 

16. Currently, siRNA therapeutics are managed as chemical drugs around the world and prefer toxicology study in NHP. Meanwhile, recommend safety evaluations in mice or rats.